\chapter{Introduction and Literature Review}

Antibiotics are chemicals that are used to treat and prevent bacterial infections.
The first antibiotics were isolated in the early 20th century from environmental bacteria and fungi and were adopted quickly into human medicine \parencite{Hutchings.2019}.
At the same time, antibiotics were used for chemotherapy, prophylaxis, and growth promotion in animal agriculture \parencite{Kirchhelle.2018}.
From a year to a couple of decades after each antibiotic reached the drug market, however, resistance was acquired in a bacterium which was historically susceptible \parencite{Ventola.2015}.
To make matters worse, we are in an antibiotic discovery void \parencite{Silver.2011}:
the most recent antibiotic drug class to be discovered, the acid lipopeptides, was reported in 1987, and novel antibiotics that have been reported since then are members of existing drug classes \parencite{Debono.1987}.

Acquired antibiotic resistance is estimated to have caused 5,400 Canadian fatalities in 2018 --- a number which is expected to rise to 13,700 deaths per year by 2050, resulting in a cumulative gross domestic product decline of \$388 billion \parencite{Finlay.2019}.
By 2050, the number of deaths globally due to multidrug-resistant microbial infections is estimated to overtake those caused by road traffic accidents and cancer combined \parencite{ONeill.2016}. Despite, the relatively recent industrialized use of antibiotics in healthcare and agriculture, antibiotic resistance is a modern crisis of ancient origin \parencite{DCosta.2011}.
To ensure the continued efficacy of our existing antibiotics, we must understand the origins of antibiotic resistance and the factors which contribute to increased antibiotic resistance in clinically relevant bacteria.

\section{Antibiotics as a global pollutant}

The mass consumption of antibiotics beginning in the mid-20th century coincides with rising antibiotic resistance in zoonotic pathogens \parencite{Kirchhelle.2018, Ventola.2015} and environmental bacteria \parencite{Madueno.2018}.
In addition to their critical role in human medicine, antibiotics are used for chemotherapy and prophylaxis in farm animals, and were historically fed \textit{en} masse to food-producing animals as growth promotion agents \parencite{Kirchhelle.2018, Witte.1998}.
The use of antibiotics as growth promotion agents has only recently been banned in several countries such as the United States in 2017 \parencite{Scott.2019}, Canada in 2018 \parencite{Finlay.2019}, and China in 2020 \parencite{Hu.2020}, but this practice still continues in many countries with few restrictions on usage \parencite{Chuanchuen.2014}.
The industrialized use of antibiotics in healthcare and agriculture continues to require mass production, which allows antibiotics to enter the environment through many pathways, including discharge from antibiotic manufacturing facilities and hospitals \parencite{Marathe.2019, Bielen.2017}, municipal sewage \parencite{Parnanen.2019}, aquaculture \parencite{Reverter.2020}, and animal agriculture \parencite{Kirchhelle.2018}.
Antibiotic pollution in the environment selects for antibiotic resistance genes \parencite{Lau.2020, Jechalke.2014, Bielen.2017, Yi.2019} which could be transferred to the human microbiome through the interconnected health of humans, animals, and the environment \parencite{Berendonk.2015, HernandoAmado.2019, Tiedje.2019, Robinson.2016}.

\section{Antibiotic resistance: A modern crisis of ancient origin}

Antibiotic resistance is ancient and ubiquitous in the environment \parencite{DCosta.2011, Dunivin.2019}.
For as long as bacteria and fungi have produced antibiotics, antibiotic resistance mechanisms were necessary as a defence against these toxins \parencite{Cundliffe.1989}.
Soil is one of the largest known reservoirs of environmental antibiotic resistance \parencite{Dunivin.2019}.
Soil bacteria are in a state of perpetual chemical warfare and use antibiotics to compete for valuable nutrients such as carbon and nitrogen but may also use them for cellular signalling \parencite{Traxler.2015, Fajardo.2008}.
The saturation of antibiotics in soil has led to an impressive arsenal of antibiotic resistance genes which are currently known to encode resistance to over a dozen antibiotic drug classes \parencite{Wright.2007, Dunivin.2019}.
Antibiotic resistance genes have been sequenced from 30,000 year-old permafrost, and some extant resistance gene families, such as serine beta-lactamases, have been predicted to share the same function as their ancestral sequences from two billion years ago \parencite{DCosta.2011, Hall.2004}.
Because of this conservation of function and continued selection due to antibiotic production, the totality of antibiotic resistance genes in soil --- the soil “resistome” --- is incredibly diverse, and can be selected for by anthropogenic antibiotic pollution \parencite{Lau.2020, Jechalke.2014}.

\section{One Health as a way forward}

“One Health” is a framework that describes the interconnectedness of human, animal, and environmental health, and has been adopted by global health organizations, nations, and researchers to help understand and mitigate the crisis of acquired antibiotic resistance \parencite{Tiedje.2019}.
In 2015, the World Health Organization released their Global Action Plan on Antimicrobial Resistance which identified an important knowledge gap of “understanding how resistance develops and spreads, including how resistance circulates within and between humans and animals and through food, water and the environment” \parencite{WorldHealthOrganization.2015}.
In this Action Plan, the World Health Organization recommended that individual member nations establish national action plans on antimicrobial resistance by adopting the One Health approach to mitigate resistance.
Canada’s Federal Framework for Action established the Canadian Antimicrobial Resistance Surveillance System (CARSS) to expand antimicrobial resistance surveillance to a national level, and in the CARSS 2020 report, the federal government acknowledged that “there is limited data regarding environmental surveillance — a necessary component of any One Health framework” \parencite{PublicHealthAgencyofCanada.2014, PublicHealthAgencyofCanada.2020}.
Of the three pillars of the One Health framework, the role of the environment in clinically relevant antibiotic resistance continues to be the least understood \parencite{Robinson.2016}.

\subsection{The shared human-soil resistome}

Under the One Health framework, anthropogenically-driven increases of antibiotic resistance in soil bacteria may pose a threat to human health due to the shared human-soil resistome \parencite{Forsberg.2012}.
The human microbiome and human bacterial pathogens share antibiotic resistance genes with environmental bacteria \parencite{Forsberg.2012, Smillie.2011, Pal.2016}, but the frequency and context of this exchange is poorly understood \parencite{Berendonk.2015, Huijbers.2015} due to the challenges associated with source attribution --- i.e. determining the exact pathway of a resistance gene from environment to the human microbiome \parencite{Tiedje.2019, Li.2018}.
In a bioinformatics analysis involving 200 soil and 100 human gut metagenomes, 25\% of gut-associated antibiotic resistance genes (\textit{n} = 12) were shared with resistance genes found in soil \parencite{Pal.2016}.
This in silico work is also supported by functional metagenomics studies which have recovered resistance genes in soil bacteria that are identical or very similar to those detected in clinical isolates \parencite{Forsberg.2012, Lau.2017b, Allen.2009}.

Antibiotic resistance genes may be transmitted from soil bacteria to the human microbiome through the consumption of produce \parencite{Maeusli.2020, Blau.2018}.
The transmission of antibiotic resistance from an environmental bacterium on a leafy vegetable, to \textit{Escherichia coli}, and then to a commensal gut bacterium has recently been demonstrated in the mouse microbiome which is a useful model for the human microbiome \parencite{Maeusli.2020, Krych.2013}.
Multi-drug resistance plasmids containing tetracycline, beta-lactam, sulfonamide, aminoglycoside, and fluoroquinolone resistance genes have also been captured in \textit{E. coli} from bacteria in cilantro and mixed salad, indicating that this process could occur in the human gut \parencite{Blau.2018}.
In addition, vegetables grown in soil enriched with antibiotic resistant bacteria can themselves be enriched with the same antibiotic resistance genes \parencite{Murray.2019, Rahube.2016, Rahube.2014}.
Overall, transmission of antibiotic resistance genes from soil bacteria to the human microbiome is plausible, but more research is needed to determine the frequency and mechanisms of this transmission.

\section{The interaction of the soil bacterial resistome and mobilome}

The soil bacterial resistome is generally considered to be structured by bacterial community composition as most antibiotic resistance genes in soil bacteria are embedded within the bacterial chromosome and are therefore inherited vertically \parencite{Dunivin.2019, Forsberg.2014}.
A high soil bacterial diversity has been proposed to “act as a biological barrier” for increased antibiotic resistance as a loss in soil bacterial species diversity is correlated with increased antibiotic resistance gene abundance \parencite{vanGoethem.2018, Chen.2019c, Vivant.2013}.
When a selective pressure (e.g. antibiotics) is strong enough, however, the soil resistome could become ‘decoupled’ from bacterial community composition and diversity as antibiotic resistance genes can be exchanged and re-arranged horizontally \parencite{Johnson.2016}.

\subsection{Mobile genetic elements and horizontal gene transfer}

Mobile genetic elements are entities that promote the mobility of DNA sequences within (chromosome–plasmid, plasmid–plasmid, chromosome–chromosome) and between bacterial genomes, and the totality of all mobile genetic elements in an environment is referred to as the “mobilome” \parencite{Partridge.2018, Perry.2013}.
The mobilome facilitates the horizontal gene transfer of antibiotic resistance genes between bacteria, and includes elements such as plasmids and transposons \parencite{Partridge.2018}, bacteriophages \parencite{Subirats.2016}; \parencite{ColomerLluch.2011}, and membrane vesicles \parencite{Chattopadhyay.2015}.
Horizontal gene transfer occurs through three main mechanisms: conjugation (physical interaction between bacteria), transformation (intake of extracellular DNA), and transduction (phage-mediated) \parencite{Partridge.2018}.
While all three of these mechanisms are known to occur in soil, conjugation has been studied the most extensively and is the most frequent mechanism of horizontal transfer of antibiotic resistance in soil \parencite{Perry.2013}, though transformation and transduction likely also play important roles \parencite{Perry.2013, Aminov.2011}.
Of all of the known non-plasmid mobile genetic elements to mobilize antibiotic resistance in soil, integrons may be the most genetically diverse \parencite{Ghaly.2019}.

\subsection{Integrons}

Integrons are mobile genetic elements that are capable of acquiring, re-arranging, and expressing antibiotic resistance genes within their environments, but notably lack the capability to move their selves, relying upon other mobile genetic elements such as plasmids and transposons for mobility \parencite{Gillings.2014}.
Integrons sample their environment for gene cassettes \parencite{Ghaly.2020} --- pieces of DNA that usually contain one open reading frame followed by a cassette-associated recombination site (\textit{attC}).
Gene cassettes carry a diverse repertoire of antibiotic resistance genes, putative virulence genes, and many other genes of unknown function that have been proposed as a discovery platform for potentially novel natural products \parencite{Ma.2017, Ghaly.2019, Ghaly.2020}.
In a recent sequencing study of the gene cassette metagenomes of soil samples from Antarctica and Australia, it was estimated that there are 4,000 to 18,000 unique gene cassettes per 0.3 g of soil \parencite{Ghaly.2019}.
\todo{Vera:
			The leading researchers in the field refer to the totality of gene cassettes in the environment as the "cassette metagenome", so I think it'd be a good idea to keep this terminology for consistency with the literature.
			E.g. \url{http://www.ncbi.nlm.nih.gov/pubmed/31948729}
}

Integrons are characterized by i) an integron-integrase gene (intI) encoding a site-specific tyrosine recombinase, ii) an integron-associated recombination site (\textit{attI}), where incoming gene cassettes are inserted with the help of IntI, and iii) an integron-associated promoter (Pc) which expresses downstream gene cassettes (Figure \ref{fig:class-1-integron-structure}) \parencite{Gillings.2014}.
IntI catalyzes the recombination of \textit{attC} with \textit{attI} to insert an incoming gene cassette downstream of Pc and can also reversibly excise an integrated gene cassette from the integron structure.
The recombination event produces two daughter molecules: a duplicate of the original integron structure, and the other with the integrated gene cassette \parencite{Ghaly.2020}.
This phenomenon allows the host bacterium to sample each gene cassette for fitness tradeoffs prior to stable integration, and in the context of a cassette-embedded antibiotic resistance gene, maintain the antibiotic resistance phenotype if it confers a selective advantage \parencite{Ghaly.2020}.

\begin{figure}[htpb]
	\centering
		\includegraphics[width=0.75\textwidth]{figures/integron/class-1-integron-structure_v02.pdf}
	\caption[Structure of a class 1 integron.]{
		A class 1 integron is characterized by its class 1 integron-integrase gene (\textit{intI1}) and a 3' conserved sequenced.
	}
	\label{fig:class-1-integron-structure}
\end{figure}

\subsection{Co-selection}

Mobile genetic elements, especially integrons, facilitate the co-selection of antibiotic resistance genes in soil \parencite{Pal.2015}.
Co-selection occurs when antibiotic exposure results in increased resistance to an environmentally absent antibiotic drug class.
Co-selection can be explained through two main processes: i) cross-resistance, when an antibiotic resistance gene is selected by an environmentally present drug class and also confers resistance to an absent drug class; and ii) co-resistance, when an antibiotic resistance gene is selected and is physically linked to a different resistance gene which confers resistance to an absent drug class \parencite{Wales.2015}.
Class 1 integrons, which are known to possess gene cassettes that are heavily biased towards conferring antibiotic resistance phenotypes \parencite{Ghaly.2020, Yang.2021}, facilitate the co-selection of antibiotic resistance genes in the environment by forming multi-drug resistance gene cassette arrays \parencite{Naas.2001}.
Furthermore, class 1 integrons form linkage clusters of antibiotic resistance in soil, as they frequently co-occur with other mobile genetic elements and with antibiotic resistance genes that are not embedded within gene cassettes \parencite{Johnson.2016, Pal.2015}.

\subsection{Class 1 integrons}

Of the hundreds of different classes of integrons \parencite{Abella.2015}, the class 1 integron is the most prolific in human pathogens and is also abundant in soil \parencite{Dawes.2010, RuizMartinez.2011, Gillings.2018}.
Class 1 integrons typically carry less than six and no more than eight gene cassettes \parencite{Gillings.2014, Naas.2001}.
Class 1 integrons are distinguished from other classes of integrons by their \textit{intI1} gene known as the class 1 integron-integrase, which are 98\% identical in amino acid sequence \parencite{Roy.2021}.
The “clinical” or “sul1-type” variant of class 1 integrons has a 3’ conserved segment with a partially deleted but semi-functional disinfectant resistance gene qacE${\Delta}$1, followed by the sulfonamide antibiotic resistance gene sul1 (Figure \ref{class-1-integron-structure}) \parencite{Partridge.2018}.
From a One Health perspective, class 1 integrons are of particular concern because i) they have become endemic to human and environmental microbiomes \parencite{Gillings.2017}, ii) they are increased in the presence of antibiotic pollution \parencite{Gillings.2017, Wright.2008, Stalder.2014}, iii) their gene cassette content is biased towards conferring antibiotic resistance phenotypes \parencite{Yang.2021}, iv) some antibiotics indirectly increase the transcriptional activity of \textit{intI1}, thereby promoting gene cassette recombination \parencite{Baharoglu.2010}, and v) they form co-occurrence linkage clusters with other mobile genetic elements and antibiotic resistance genes \parencite{Pal.2015}.
Class 1 integrons are known to be enriched in soils that have been polluted with macrolide antibiotics \parencite{Lau.2020}.

\section{Macrolide antibiotics}

\subsection{Importance to human and animal medicine}

Macrolide antibiotics are the third most-consumed antibiotics in Canada and are used as first-line treatments for serious diseases such as community acquired pneumonia (\textit{Streptococcus} and \textit{Mycoplasma}), campylobacteriosis (\textit{Campylobacter jejuni}, \textit{C. coli}), and as alternatives for individuals allergic to beta-lactams \parencite{PublicHealthAgencyofCanada.2020, CapeloMartinez.2019}.
Despite their prolific use in human medicine, most macrolide antibiotics that are sold are consumed by food-producing animals for chemotherapy and prophylaxis \parencite{CapeloMartinez.2019}.
In 2018 alone, 87,221 kg of macrolide antibiotics were sold for consumption in Canadian agriculture \parencite{PublicHealthAgencyofCanada.2020}.
These antibiotics have been deemed “critically important” for human medicine by the World Health Organization and resistance to these drugs is rising \parencite{Resistance.2017, PublicHealthAgencyofCanada.2020}.
Risk management strategies that focus on reducing macrolide presence in the environment will mitigate future risks to human health.

\subsection{Structure}

\begin{figure}[htb]
	\centering
		\includegraphics[width=0.7\textwidth]{figures/macrolide-chemical-structures/macrolide-chemical-structures.png}
	\caption{Chemical structures of erythromycin, clarithromycin, and azithromycin.}
	\label{fig:macrolide-chemical-structures}
\end{figure}

Erythromycin A was first isolated from the soil bacterium \textit{Saccharopolyspora erythraea} in 1952 and most other macrolide antibiotics are chemically modified derivatives of erythromycin A, which is the primary active compound in the antibiotic medicine erythromycin \parencite{Haight.1952}.
Erythromycin, clarithromycin, and azithromycin are the most consumed macrolides in human medicine, as reflected by their prevalence in wastewater \parencite{Miao.2004, RodriguezMozaz.2020}.
Macrolide antibiotics are characterized by a 14-, 15-, or 16-membered macrocyclic lactone ring bound to at least one deoxy sugar (erythromycin A, clarithromycin, and azithromycin are bound to desosamine and cladinose) (Figure \ref{fig:macrolide-chemical-structures}) \parencite{CapeloMartinez.2019}.
Clarithromycin is identical to the 14-membered erythromycin A but with a methylated C6-hydroxy group, resulting in a more acid-labile molecule.
Azithromycin is a 15-membered macrolide created from the insertion of a nitrogen atom into the lactone ring of erythromycin A, resulting in more potent antibacterial activity against many gram-negative pathogens such as \textit{Haemophilus influenzae} (bacterial flu) and \textit{Neisseria gonorrhoeae} (gonorrhea) \parencite{Yanagihara.2009}.

\subsection{Mechanisms of action and resistance}

Macrolides inhibit protein synthesis in gram-positive (and some gram-negative) bacteria by reversibly binding to the 23S ribosomal RNA (rRNA) within the bacterial 50S ribosomal subunit, at the entrance of the peptide exit tunnel, which imperfectly prevents assembly and elongation of the peptide \parencite{CapeloMartinez.2019, Fyfe.2016}.
This mechanism is usually bacteriostatic --- the macrolides alone do not kill all of the bacterial cells and the host’s immune system must clear the remainder of the infection \parencite{Pankey.2004}.
Macrolide antibiotic resistance mechanisms in bacteria are diverse \parencite{Fyfe.2016}.
Resistance can be evolved through target site mutation in the ribosome or can be horizontally acquired:
Antibiotic resistance genes may encode a methyltransferase which methylates the ribosome and prevents binding of the antibiotic (erm gene family), or an efflux pump to remove the antibiotic from the cell (msr and mef gene families), or a phosphotransferase to inactivate the antibiotic (mph gene family) \parencite{Fyfe.2016}.
Many of these antibiotic resistance genes are mobile as demonstrated by the erm gene family, as over 40 erm genes have been identified and most of them are plasmid-encoded \parencite{Alcock.2020, Leclercq.2002}.

\subsection{Effects of long-term macrolide antibiotic pollution in agricultural soil}

Macrolide antibiotic pollution of soil is known to promote antibiotic resistance \parencite{Lau.2020}.
Over an eight-year period, soil field plots were annually exposed to the macrolide antibiotics erythromycin, clarithromycin, and azithromycin which resulted in increased abundances of antibiotic resistance genes and mobile genetic elements, including class 1 integrons.
Interestingly, most of the antibiotic resistance genes that were increased were predicted to confer resistance to non-macrolide antibiotic drug classes, indicating that macrolide antibiotic exposure of soil co-selects for resistance to aminoglycosides, sulfonamides, and trimethoprim.
Several of these antibiotic resistance genes are known to be associated with class 1 integrons, suggesting a role for class 1 integrons in this co-selection process \parencite{Lau.2020}.
Macrolide antibiotics are also more rapidly degraded in soil with a previous exposure history to macrolides, indicating that macrolides may have an effect on soil microbial diversity and composition \parencite{Topp.2016}.
This effect could be ecotoxic in nature and could represent a threat to agricultural productivity \parencite{Prashar.2014}.

\section{Biosolids as a vector for macrolide antibiotic pollution of soil}

Macrolide antibiotics are discharged into the environment through human waste and are inefficiently removed by most wastewater treatment processes \parencite{LeMinh.2010, Luo.2014}.
In Canada, only 28\% of the population is served by tertiary wastewater treatment which removes greater quantities of macrolides than other treatments, and the focus of this treatment is on disinfection rather than the removal of pharmaceuticals \parencite{EnvironmentandClimateChangeCanada.2020, LeMinh.2010}.
Abundances of antibiotics and antibiotic resistance genes are currently unregulated in Canadian wastewater effluent and many other countries, and as a result, wastewater effluent is also a hotspot of antibiotic resistance genes and mobile genetic elements \parencite{Rizzo.2013, Che.2019}.
Macrolide antibiotics from wastewater effluent can contaminate soil through the agricultural use of treated sewage sludge \parencite{McClellan.2010, Sabourin.2012}.

\subsection{Agricultural use of biosolids}

Biosolids (treated sewage sludge) are recycled material from wastewater treatment plants that can be used as an agricultural fertilizer and soil amendment \parencite{Sharma.2017}; the solid portion of biosolids is comprised of approximately 50\% organic matter and 50\% mineral material \parencite{OntarioMinistryofAgricultureFoodandRuralAffairs.2010}.
Unfortunately, antibiotics that survive the wastewater treatment process can carry over into biosolids, including those of the macrolide antibiotics drug class such as erythromycin, clarithromycin, and azithromycin \parencite{McClellan.2010, Sabourin.2012, Chenxi.2008}.
Biosolids are produced from the separation of wastewater into water and solids, followed by treatment of the solid portion to reduce pathogens and odour using a combination of chemical, biological, or physical processes \parencite{LeMinh.2010}.
Biosolids improve soil quality and fertility:
soil that is more fertile requires less inorganic fertilizer, which reduces the risk of fertilizer runoff into adjacent water sources, and soil that has more organic matter has increased moisture retention.
Biosolids are applied to agricultural soil on every continent except Antarctica, but usage is highly variable:
almost all of the biosolids that are produced in the United Kingdom (78\%) and Ireland (96\%) are land-applied, whereas only 55\% are land-applied in the United States \parencite{Sharma.2017}.
There are concerns, however, that the long-term application of biosolids to agricultural soil could introduce macrolide antibiotics into the environment and promote resistance in soil bacteria, which could be transferred to humans via consumption of produce under the One Health framework \parencite{Lau.2020, Sabourin.2012}.

\subsection{Concentrations of macrolide antibiotics in biosolids and comparison to PNEC}

In a survey of 74 locations producing treated biosolids in the United States, the 95\textsuperscript{th} percentile concentrations of detected macrolides were 0.12 mg kg\textsuperscript{-1} biosolids (dry weight) erythromycin, 0.17 mg kg\textsuperscript{-1} clarithromycin, and 3.17 mg kg\textsuperscript{-1} azithromycin \parencite{U.S.EnvironmentalProtectionAgency.2021}.
These concentrations are 100-fold, 680-fold, and 1,268-fold greater than the Predicted No-Effect Concentrations (PNEC) for these antibiotics in freshwater as determined by \cite{BengtssonPalme.2016}.
The PNEC is the concentration above which antibiotic resistance could be selected for in environmental bacteria and have been proposed as limits for the regulation of antibiotics in the environment.
The 95\textsuperscript{th} percentile concentrations of macrolides in biosolids greatly exceed the PNECs for \textcolor{red}{freshwater}, and biosolids could therefore realistically select for antibiotic resistance in land-applied soil.

\subsection{Critical knowledge gaps}

The land-application of biosolids introduces antibiotics into agricultural soil that have carried over from the wastewater treatment process \parencite{McClellan.2010, Sabourin.2012}.
These antibiotics are present at concentrations that are predicted to select for resistance in the soil bacterial community \parencite{U.S.EnvironmentalProtectionAgency.2021, BengtssonPalme.2016}, and the exposure of soil to macrolide antibiotics increases the abundance of antibiotic resistance genes and mobile genetic elements in soil bacteria, including class 1 integrons \parencite{Lau.2020}.
This antibiotic exposure is also known to co-select for resistance to anthropogenically absent drug classes of antibiotics and resistance genes that are known to be associated with class 1 integron gene cassettes \parencite{Lau.2020}.

The effects of macrolide antibiotic exposure on the soil bacterial community and the integron gene cassette metagenome remain to be determined, as does the potential for macrolides to select for antibiotic resistance in soil bacteria at concentrations that are environmentally relevant to a biosolids exposure scenario.
Because the health of humans and soil are interconnected under the One Health framework \parencite{Tiedje.2019}, and because biosolids are a vector for the introduction of macrolide antibiotics into the environment \parencite{Sabourin.2012, McClellan.2010}, we must determine the consequences of long-term macrolide exposure on the development of antibiotic resistance in soil bacteria in order to assess if the repeated use of biosolids in agriculture may pose a risk to human health.

\section{Review of sequencing-based methods}

\subsection{16S rDNA sequencing}

Most soil bacteria are uncultivable:
of the approximately 108 cells of bacteria that can be found in a single gram of bulk soil, less than 1\% are estimated to be cultivable using standard growth techniques \parencite{Raynaud.2014, vanPham.2012}.
The selectivity of nutrient media, competition in media by faster growing organisms, and low abundance in the environment relative to other species all contribute to the difficulty in culturing most soil bacteria \parencite[229–300]{vanElsas.2019b}.
Sequencing-based approaches to investigate the bacterial community have shone a light on the incredible diversity of uncultivable soil bacteria \parencite{Hug.2016} and have allowed researchers to investigate the responses of the soil bacterial community to environmental perturbations \parencite{Isobe.2019, Isobe.2020}.
Of the different sequencing-based approaches available to investigate soil bacterial community composition, 16S rDNA sequencing and metagenomic sequencing are presently the most common.

16S rDNA sequencing involves the targeted amplicon sequencing of the 16S rRNA gene.
The bacterial 16S rRNA gene is used to determine bacterial taxonomy due to i) regions of highly conserved sequence between bacterial species and ii) hypervariable regions which allow for species-specific classification \parencite{vanPham.2012}.
Typically, only a subset of the hypervariable regions are sequenced (usually some combination of the V3, V4, V5, V6 regions) to classify bacterial taxa \parencite{Yang.2016}, though advances in long-read technologies have made full-length 16S rDNA sequencing an attractive alternative \parencite{Shin.2016, Numberger.2019}.
First, total genomic DNA is isolated from the soil sample and the hypervariable regions (the 16S rDNA) of the bacterial 16S rRNA gene are PCR amplified using site-specific primers.
Next, a DNA library is prepared from the resulting amplicons and the library is subsequently sequenced.
Finally, the biological sequence data that are generated from the sequencer are analyzed using bioinformatics software.
Taxonomic classification software such as QIIME 2 can cluster sequence reads based on dissimilarity thresholds into operational taxonomic units, or software such as DADA2 can attempt to infer biological sequences prior to PCR and sequencing to construct amplicon sequence variants \parencite{Bolyen.2019, Callahan.2016}.
The use of amplicon sequence variants over operational taxonomic units is preferred, as sequence variants attempt to deal with sequencing errors and better reflect the DNA that was actually sequenced \parencite{Callahan.2017}.

\subsection{Metagenomic sequencing}

Metagenomic sequencing, the non-selective sequencing of the total genomic DNA in an environment, is another popular approach for determining bacterial community composition in soil.
In metagenomic sequencing, total genomic DNA is isolated from the soil, a DNA library is prepared from the total genomic DNA, and the DNA library is then sequenced.
Metagenomic sequencing has several advantages over 16S rDNA sequencing for determining bacterial community composition:
16S rDNA sequencing suffers from primer bias during PCR, as the primers amplify different ribosomal sequences with different efficiencies, resulting in a bias of sequence reads to taxa with rRNA genes that are more similar to the primer-binding site \parencite{Tremblay.2015}.
In addition, metagenomic sequencing generates data covering multiple genes and possibly entire bacterial genomes, allowing for a metagenomic functional analysis in addition to taxonomic analysis \parencite{Li.2015c}.
At present, the greatest downside to metagenomic sequencing is the higher financial cost associated with metagenomic sequencing compared to 16S rDNA sequencing as a greater sequencing depth is required in order to achieve a detailed picture of the bacterial community \parencite{Scholz.2012}.

Metagenomic sequencing can also be used to identify antibiotic resistance genes and mobile genetic elements within a bacterial community \parencite{Boolchandani.2019}.
Metagenomic sequencing confers many advantages over other methods for studying antibiotic resistance in soil bacteria.
Antibiotic resistance genes and mobile genetic elements are distributed among diverse soil bacterial taxa --- many of which are difficult to cultivate under normal laboratory conditions \parencite{Dunivin.2019}.
Metagenomic sequencing, compared to PCR-based methods, also allows for the discovery of novel antibiotic resistance genes and mobile genetic elements for which PCR primers have not been developed or are not available \parencite{Boolchandani.2019}.
Antibiotic resistance genes and mobile genetic elements can be identified in metagenomic sequence data by aligning sequence reads to databases of known antibiotic resistance genes and mobile genetic elements, such as the Comprehensive Antibiotic Resistance Database (\gls{CARD}) \parencite{Alcock.2020}.
Other bioinformatics software, such as Metagenomic Markov models for Antimicrobial Resistance Characterization (\gls{Meta-MARC}), use machine learning principles to identify novel antibiotic resistance genes from metagenomic sequence data \parencite{Lakin.2019}.

\subsection{Integron gene cassette sequencing}

Integrons can be identified in metagenomic datasets using software that scans for \textit{intI1} and \textit{attC} sites \parencite{Cury.2016}.
Such software could theoretically be fine-tuned to only target specific classes of integrons or could be made more sensitive to detect novel classes of integrons.
However, the analysis of metagenomic data alone is unlikely to capture the full diversity of integron gene cassettes in a soil sample due to the complexity of the microbiome.
In addition, using sequence alignment software such as BLAST or DIAMOND to search for class 1 integrons by identifying \textit{intI1} wouldn’t capture the diversity of the hundreds of other known classes of integrons \parencite{Altschul.1990, Buchfink.2015}.
The targeted amplicon sequencing of integron gene cassettes is a PCR-based approach that can be used to characterize the diversity of integron gene cassettes in any environment:
PCR primers can be designed to target the \textit{attC} or \textit{attI} sites and/or the integron-integrase gene to amplify gene cassettes within a specific integron class or within diverse environmental integrons, and similar to 16S rDNA amplicons, these amplicons can then be sequenced and analyzed using bioinformatics software for antibiotic resistance gene identification \parencite{Yang.2021, Ghaly.2019}.
Cassette-embedded genes could also be assigned more general functions using databases of orthologous groups such as eggNOG \parencite{HuertaCepas.2019}.

\subsection{Compositional data analysis}

Much statistical software has been developed to help identify biologically meaningful differences in the diversity and compositions of groups from sequence data.
For example, DESeq and edgeR both accept a matrix of samples versus counts as input (also known as a feature table) and then attempt to identify differentially abundant features between groups of samples (e.g. treatments) in the table \parencite{Anders.2010, Robinson.2010}.
This feature table could describe the counts of any genomic feature of interest, including bacterial amplicon sequence variants, antibiotic resistance genes, mobile genetic elements, or cassette-embedded genes.
DESeq and edgeR both assume that sequence reads can be normalized based upon sequence depth (conversion of counts to proportions); however, sequence data is compositional by nature, as sequencing instruments have constrained capacities to sequence samples, and therefore generate counts that can themselves be described as proportions of a constrained, unknown sum \parencite{Gloor.2017}.
More recently, bioinformatics tools such as ALDEx2 and ANCOM with Bias Correction (ANCOM-BC) have been developed which use statistical techniques that are appropriate for identifying differentially abundant features in sequencing datasets \parencite{Fernandes.2014, Lin.2020}.
This software can be used to investigate differences in the compositions of sequence datasets that are relevant to the analysis of soil microbiomes.

\section{Objectives and hypotheses}

Following the observed increased abundances of antibiotic resistance genes and mobile genetic elements in agricultural soil that had been annually exposed to macrolide antibiotics for eight years, the contributions of co-selection and bacterial community composition to these increases remained to be determined, as did the potential for these effects to occur at an environmentally realistic dose for a biosolids land-application scenario \parencite{Lau.2020}.
To further investigate if macrolide antibiotic exposure of soil promotes resistance at an environmentally realistic dose, and to elucidate the mechanisms of increased resistance at an effect-inducing unrealistically high dose, we obtained soil DNA from field plots treated with macrolide antibiotics for ten years and from untreated plots.
The 16S rDNA and class 1 integron gene cassettes were PCR amplified and sequenced, and the total soil metagenome was sequenced.

I hypothesized that long-term macrolide antibiotic exposure of agricultural soil, at both a realistic dose (0.1 mg kg-1 soil) and an unrealistically high dose (10 mg kg-1) for biosolids carryover, would affect the composition and diversity of the soil bacterial community, resistome, and mobilome.

I predicted that:

\begin{enumerate}
  \item{Antibiotic resistance genes and mobile genetic elements would increase in response to antibiotic exposure,}
  \item{Bacterial community composition and diversity would differ between antibiotic-exposed and -unexposed soil, and}
  \item{Integron gene cassette composition and diversity would differ between antibiotic-exposed and -unexposed soil.}
\end{enumerate}
