\chapter{Discussion}

The purpose of this project was to investigate the effect of long-term macrolide antibiotic exposure on the soil bacterial community, resistome, and mobilome --- and more specifically, to determine if an environmentally realistic dose of antibiotics for a biosolids exposure scenario could promote clinically relevant antibiotic resistance in soil bacteria.
Human and environmental health are interconnected under the One Health framework, and antibiotic resistance in soil bacteria may affect antibiotic resistance in the human microbiome or in pathogens.

\todo[color=blue!20]{I decided to present the no-effect results for the realistic dose first since I believe they're the most important for informing policy.}

\section{Realistic antibiotic exposure does not affect the diversity or composition of the soil bacterial community, resistome, or mobilome}

\todo[color=red!20]{Ed:
I re-introduced the 'realistic' and 'unrealistic' terminology back into some of the summarizing statements in this Discussion.
Is this ok?
}

Overall, we detected no effect of environmentally realistic antibiotic exposure (low dose, 0.1 mg kg\textsuperscript{-1}) on the diversity or composition of the soil bacterial community, resistome, or mobilome.
This dose is similar to what would be expected in soil following the land-application (1\% dw dw\textsuperscript{-1}) of municipal biosolids containing 95\textsuperscript{th} percentile concentrations of erythromycin, clarithromycin, and azithromycin antibiotics.
The absence of a treatment effect for any of these endpoints indicates that repeated annual application of biosolids in agriculture is unlikely to promote antibiotic resistance in agricultural soil at levels that would be of-concern to human health.

\todo[color=red!20]{Ed: Do you think the above sentence is a fair statement? Could we be challenged by assuming to know what levels of antibiotic resistance are/aren't relevant to human health in the environment? I just want to make sure I'm not overstepping in my conclusions here.}

Only three antibiotic resistance genes had increased relative abundances in response to antibiotic exposure at the low dose, and two of them are known to be associated with resistance to macrolide antibiotics (Figure \ref{fig:effect-sizes-sig-args}).
\textit{mexL} and \textit{mexP} are members of the \textit{mex} gene family which are components of chromosomally encoded efflux pumps in \textit{Pseudomonas} spp. \parencite{Mima.2005, Chuanchuen.2005}.
The \textit{mexL} gene encodes a repressor for \textit{mexJK} transcription, which are members of the efflux pump-encoding \textit{mexJK-OpmH} (triclosan resistance) and \textit{mexOprM} (macrolide, tetracycline resistance) operons \parencite{Chuanchuen.2005}.
The \textit{mexP} gene encodes the membrane fusion protein for the multi-drug efflux pump that is encoded by the \textit{MexPQ-OpmE} operon, which is known to confer resistance to several antibiotic drug classes including macrolides \parencite{Mima.2005}.
The other antibiotic resistance gene that was increased in the low dosed-soil was \textit{aac(6')-IIa}, which encodes an aminoglycoside acetyltransferase that is distributed among a variety of gram-negative pathogens, including \textit{Pseudomonas} spp., and is carried by plasmids and integrons \parencite{Shaw.1989, Partridge.2009}.
Only one mobile genetic element variant, \textit{tnpA}, was increased in the low-dosed soil (Figure \ref{fig:effect-sizes-sig-mges}).
The \textit{tnpA} gene encodes a transposase which is involved in the mobilization of bacterial mobile genetic elements such as transposons and insertion sequences across a broad host range \parencite{Partridge.2018}.

The increased abundances of a few antibiotic resistance genes in the low-dosed soil and that of \textit{tnpA} could be due to the increased abundance of a particular taxon that is intrinsically resistant to macrolide antibiotics, such as the human pathogen \textit{Pseudomonas aeruginosa}, though we did not detect any taxa that were both significantly increased and known to carry this assortment of genes.
These results are in clear contrast to the high dose of macrolide antibiotics.

\section{Unrealistically high antibiotic exposure alters the diversity and composition of the soil bacterial resistome and mobilome}

At a high dose of macrolide antibiotics (10 mg kg\textsuperscript{-1}) --- approximately 100-fold greater than the concentrations of macrolides that would be expected to result from land-application of municipal biosolids --- we detected significant effects on the diversity and composition of the soil bacterial resistome and mobilome.
While this dose is considered unrealistically high for soil receiving biosolids with 95\textsuperscript{th} percentile concentrations of macrolides, the maximum azithromycin concentration detected in the biosolids themselves (5 mg kg\textsuperscript{-1}) was within an order-of-magnitude of the high dose \parencite{U.S.EnvironmentalProtectionAgency.2009}.
This high dose of macrolide antibiotics has also been detected in sediments surrounding macrolide antibiotic manufacturing facilities \parencite{GonzalezPlaza.2019}.
Therefore, while we note that this dose is unrealistically high for a biosolids land-application scenario, comparable doses are certainly seen under other environmental contexts, and the findings of this experiment may be useful for predicting antibiotic resistance in other macrolide-contaminated environments.

The high dose of macrolide antibiotics significantly increased the number of unique antibiotic resistance genes that were detected within the soil metagenome (Figure \ref{fig:chao1-richness}a).
The majority of antibiotic resistance genes that were detected in the metagenome were detected within all three groups (control, low, and high), but approximately twice as many unique antibiotic resistance genes were detected within the high treatment group alone than the control and low groups alone (Supplementary Figure \ref{supp-fig:args-mges-compartment-size}a).
The most likely explanation for the increased number of unique antibiotic resistance genes in the high-dosed soil is selection for antibiotic resistance genes that were below the detection limit in the control and low groups, but were raised above the limit of detection by high antibiotic exposure.

In addition, a high dose of macrolide antibiotics changed the composition of antibiotic resistance genes within the soil metagenome (Figure \ref{fig:pca-ordplots-args-mges-integrons}a), and this effect extended to when antibiotic resistance genes were grouped by several drug classes (Figure \ref{fig:pca-ordplots-drug-classes}).
These differences in composition indicated that the relative abundances of several antibiotic resistance genes and target drug classes were more similar within the high-dosed soil than in the control and low-dosed soil.
The altered composition of antibiotic resistance genes in the high-dosed soil was driven by increased relative abundances of 21 antibiotic resistance genes in the high dose --- only two of which are known to confer resistance to macrolide antibiotics.
The increased relative abundances of non-macrolide antibiotic resistance genes strongly suggests co-selection via co-resistance, which could be facilitated by mobile genetic elements.

Co-resistance to different drug classes of antibiotics can occur due to the genetic linkage of antibiotic resistance on mobile genetic elements such as class 1 integrons, or when antibiotic resistance genes are carried within the same host \parencite{Pal.2015}.
Of the 21 increased antibiotic resistance genes in the unrealistically high-dosed soil metagenome, eight are known to be associated with class 1 integrons (\textit{sul1}, \textit{aac(3)-Ib}, \textit{aadA}, \textit{aadA15}, \textit{aadA22}, \textit{aadA24}, \textit{dfrA17}, \textit{dfrA15}), which agrees with the increased relative abundance of \textit{intI1} and \textit{qacE$\Delta$1} (a component of the 3' conserved sequence) in the high dose \parencite{Partridge.2009, Yan.2006, Herrero.2008}.
All of these antibiotic resistance genes have been detected in gram-negative human pathogens \parencite{Alcock.2020}.

Despite the increased relative abundances of several gene cassette-associated antibiotic resistance genes (except \textit{sul1}, which is a member of the 3' conserved sequence), integron gene cassette richness (Figure \ref{fig:chao1-richness}c) and composition (Figure \ref{fig:pca-ordplots-args-mges-integrons}c) were unaffected by antibiotic exposure.
This may be because the transcription of \textit{intI1}, which is responsible for acquiring, re-arranging, and excising gene cassettes, is regulated by the bacterial SOS response, and macrolide antibiotics (along with other protein synthesis-inhibiting drug classes) do not induce this response, thus the richness and composition would be unchanged \parencite{Baharoglu.2010}.

\todo[color=orange!30]{Vera: Does this make sense to you? Could integron gene cassette-embedded antibiotic resistance genes be amplified in response to antibiotic exposure, detected in the metagenomic analysis, but not detected in the gene cassette analysis?}

Of the remaining antibiotic resistance genes, five are known to be carried on plasmids in human pathogens (\textit{sul2}, \textit{aac(6')-Ib7}, \textit{pp-flo}, \textit{mphE}, \textit{ant(3'')-IIa}), \textit{tet(33)} is carried by the insertion sequence IS6100, and \textit{aph(3’’)-Ib} / \textit{strA} is carried on several mobile genetic elements including plasmids and transposons \parencite{Alcock.2020, Tauch.2002}.
The remaining genes with increased relative abundances are known to be chromosomally-encoded in \textit{Pseudomonas} spp. (\textit{aph(3’)-IIb}, \textit{oprN}, \textit{mexH}, \textit{triC}, \textit{mexQ}) or in \textit{Burkholderia} spp. (\textit{opcM}) \parencite{Hachler.1996, Mesaros.2007, Mima.2005, Mima.2007, Burns.1996}.
All of these remaining genes with the exception of \textit{aph(3')-IIb}, an aminoglycoside phosphotransferase, encode components of antibiotic efflux pumps.
Overall, of the 21 increased antibiotic resistance genes in the high-dosed soil, 15 are known to be carried by mobile genetic elements and all are known to be associated with human pathogens.

The high dose of antibiotics similarly increased the number of unique mobile genetic element variants (Figure \ref{fig:chao1-richness}b) and altered the composition of the mobilome (Figure \ref{fig:pca-ordplots-args-mges-integrons}).
More mobile genetic element variants were detected in the high-dosed soil group alone (\textit{n} = 119) than were shared between any combination of the other groups, suggesting that the high dose of macrolides raised many mobile genetic element variants over the limit of detection (Supplementary Figure \ref{supp-fig:args-mges-compartment-size}b).
Of the 23 mobile genetic element variants with increased relative abundances in the high-dosed soil (Figure \ref{fig:effect-sizes-sig-mges}), most were \textit{tnpA} variants (\textit{n} = 15), which suggests that some of the increased antibiotic resistance genes may have been mobilized by transposons or insertion sequences.
The exact mechanism of co-selection of macrolide and non-macrolide antibiotic resistance genes in the high-dosed soil could not be elucidated.
However, because antibiotic resistance genes associated with several types of mobile genetic elements were increased, and several mobile genetic elements were increased, it's plausible that multiple co-selection processes were active in the high-dosed soil simultaneously.

\section{Antibiotic exposure enriches for fastidious taxa}

In this study, we detected increased relative abundances of three bacterial taxa in the low-dosed soil and two taxa in high-dosed soil (Table \ref{table:effect-sizes-sig-taxa}).
For the low-dosed soil, the effect sizes for two of the three increased taxa were relatively low ($W$ < 5), but an unknown Subgroup 6 Acidobacterium taxon was over 45-fold more abundant in the low-dosed soil than the control.
This taxon was present in both antibiotic treated groups but not in the control soil.

\todo[color=blue!20]{Should I capitalize "Acidobacterium"? Similarly, "Cyanobacterium"?}

Acidobacteria are a largely uncultivated, highly abundant bacterial phylum in Canadian agricultural soil and play an important role in shaping the soil bacterial community through their decomposition of organic carbon \parencite{Solden.2016, Banerjee.2016, Banerjee.2016b}.
Furthermore, Acidobacteria are a known reservoir of macrolide antibiotic resistance in urban surface waters through their expression of the \textit{erm} gene family, and have been reported to be increased in macrolide-polluted sediments, suggesting intrinsic macrolide resistance among some taxa \parencite{Yi.2019, Milakovic.2020}.
Conservatively, the unknown Subgroup 6 Acidobacterium taxon may represent a macrolide-resistant decomposer, but more speculatively, could represent a species that is able to use macrolides as an alternative source of carbon.
Further studies would be required to investigate the macrolide biodegradation potential of this Acidobacteria taxon.

For the high-dosed soil, the effect sizes for both increased taxa were high ($W$ $>$ 20) and both were identified as unknown Gitt-GS-136 Chloroflexi spp. (Table \ref{table:effect-sizes-sig-taxa}).
Chloroflexi are fastidious bacteria with diverse metabolisms and, like Acidobacteria, are a known reservoir of macrolide resistance in the environment, though their overall role in environmental antibiotic resistance is still poorly understood \parencite{Gupta.2013, Islam.2019, Yi.2019}.
To our knowledge, this phylum has not been reported to carry any of the antibiotic resistance genes that were increased in this study, though the role of Chloroflexi in antibiotic resistance remains understudied \parencite{Razavi.2017}.

Although we did not detect a significant effect of macrolide antibiotic exposure at either dose on the soil bacterial community composition or diversity, some taxa do respond to exposure.
In a previous investigation of the persistence of macrolide antibiotics in soils that were annually exposed to a low or high dose of erythromycin, clarithromycin, and azithromycin for five years, or were left untreated, macrolide antibiotics were degraded more rapidly in the soils with an exposure history to macrolides than in the untreated control soil \parencite{Topp.2016}.
It is possible that Acidobacteria or Chloroflexi may have played a role in the accelerated biodegradation of these macrolide antibiotics.

\section{Unrealistically high antibiotic exposure decreases Cyanobacteria abundance}

The only bacterial phylum that was differentially abundant in response to antibiotic exposure was Cyanobacteria (Figure \ref{fig:effect-sizes-sig-phyla-metagenomic}).
The relative abundance of Cyanobacteria was decreased in the high-dosed soil but not in the low-dosed soil, and this effect was observed only in the metagenomic analysis and not in the 16S rDNA analysis (Supplementary Figure \ref{supp-fig:effect-sizes-sig-phyla-16S}).

The detection of this treatment effect in only one of two taxonomic analyses may be due to differences in how bacterial taxa are assigned a taxonomic identity between these two approaches.
For the metagenomic analysis, MetaPhlAn3 was able to achieve species-level taxonomic classification by matching the metagenomic sequence reads to a database of clade-specific marker genes \parencite{Beghini.2020}.
For the 16S rDNA analysis, DADA2 established amplicon sequence variants from the 16S rDNA sequence data based upon predictions related to sequencing error, and these sequence variants were then assigned taxonomy using a feature classifier that was trained on a 16S rRNA gene database \parencite{Callahan.2016, Bokulich.2018, Quast.2013}.
One issue that remains unresolved with 16S rDNA sequencing is that different bacterial genomes have different copy numbers of the 16S rRNA gene, and there is currently no good method for correcting relative abundances obtained from 16S rDNA sequencing for 16S rRNA copy number variation \parencite{Starke.2021}; therefore, the relative abundances obtained from 16S rDNA sequencing are biased towards bacterial genomes with greater copy numbers of the 16S rRNA gene.
Metagenomic sequencing is not without its own inherent biases, however \parencite{McLaren.2019}.
Metagenomic sequencing and 16S rDNA sequencing produce different results centered around the biological truth of the bacterial community, and disagreement between statistically significant differences should be expected but interpreted with caution.

Cyanobacteria have historically been considered as indicator species for antibiotic pollution of aquatic ecosystems by regulatory agencies due to their sensitivity to several drug classes of antibiotics \parencite{CommitteeforMedicinalProductsforHumanUse.2006, LePage.2017}, but this response is not uniform across all species and to all antibiotics \parencite{LePage.2017, Dias.2015}.
For example, the minimum inhibitory concentration (the lowest concentration preventing visible growth) of the cyanotoxin-producing cyanobacterium \textit{Microcystis aeruginosa} to $\beta$-lactam antibiotics can be as low as 0.1 mg L\textsuperscript{-1}, while the minimum inhibitory concentrations of $\beta$-lactams for the tropical Cyanobacteria \textit{Gloeocapsa} sp. and \textit{Chroococcidiopsis} sp. may be 100-fold greater \parencite{Dias.2015, Reynaud.1986}.
The decreased relative abundance of Cyanobacteria in the high-dosed soil of this present study likely represents an environmental transition from a dose below the minimum inhibitory concentration of macrolide antibiotics to a dose above this concentration for at least some Cyanobacteria.
The minimum no-observed-effect concentrations \textcolor{red}{(on growth inhibition)} of azithromycin and erythromycin for Cyanobacteria were reported to be at-most 0.0015 and 0.0062 mg L\textsuperscript{-1}, which are approximately 20--70-fold lower than the concentration of macrolides in the low-dosed soil, and 1,600--6,700-fold lower than the concentration of macrolides in the high-dosed soil \parencite{LePage.2019}.
Therefore, the decreased abundance of Cyanobacteria in the high-dosed soil of this present study is in agreement with the known no-observed-effect concentrations for erythromycin and azithromycin.
Our inability to detect an effect of macrolide antibiotic exposure at the low dose may be due to a higher minimum inhibitory concentration for soil Cyanobacteria or insufficient sensitivity to detect this effect using metagenomic sequencing.

\section{High antibiotic exposure decreases resistance to $\beta$-lactams}

Of the seven antibiotic resistance genes that were decreased in response to macrolide antibiotic exposure (five in the high dose, two in the low dose), all were predicted to encode resistance to $\beta$-lactam antibiotics (Figure \ref{fig:effect-sizes-sig-args}).
$\beta$-lactam antibiotics are bactericidal against both gram-negative and gram-positive bacteria by inhibiting synthesis of the cell wall, thereby leading to lysis and cell death \parencite{CapeloMartinez.2019}.
The $\beta$-lactam drug class of antibiotics was among the first to be brought to the drug market with the discovery of penicillin in 1928 by Alexander Fleming \parencite{Fleming.1929}.
The subsequent industrialized production and mass consumption of penicillins by the mid-1940's has resulted in increased acquired resistance to $\beta$-lactams, especially due to methicillin-resistant strains of \textit{Staphylococcus aureus} \parencite{PublicHealthAgencyofCanada.2020}.

$\beta$-lactam resistance genes are highly abundant in soil bacteria, even in the absence of anthropogenic antibiotic pollution, and over 90\% of these genes are encoded chromosomally \parencite{Dunivin.2019, vanGoethem.2018, Mindlin.2017}.
Of the $\beta$-lactam resistance genes that were decreased, two SHV-family $\beta$-lactamase encoding genes (\textit{bla}\textsubscript{SHV-71}, \textit{bla}\textsubscript{SHV-165}), one CTX-M $\beta$-lactamase (\textit{bla}\textsubscript{CTX-M-117}), one PEDO-family metallo-$\beta$-lactamase (\textit{bla}\textsubscript{PEDO-1}), and one \textit{ampC}-type $\beta$-lactamase (\textit{E. coli ampC}) were decreased in the high-dosed soil, while two TEM-family $\beta$-lactamase encoding genes (\textit{bla}\textsubscript{TEM-1}, \textit{bla}\textsubscript{TEM-22}) were decreased in the low-dosed soil.
\textit{bla}\textsubscript{TEM-1} was the first plasmid-associated $\beta$-lactam resistance gene to be identified and has since spread throughout gram-negative pathogens (e.g. \textit{Acinetobacter baumanii}, \textit{E. coli}, \textit{Klebsiella pneumoniae}).
Other members of the TEM-family of $\beta$-lactamase genes, such as \textit{bla}\textsubscript{TEM-22}, have a more narrow host range but confer resistance to extended-spectrum $\beta$-lactams (able to hydrolyze oximino-cephalosporins) \parencite{Bradford.2001, Garlet.1993}.

The most likely explanation for the decreased abundances of $\beta$-lactam resistance genes in the macrolide antibiotic-exposed soil is the decreased abundance of macrolide-susceptible bacteria carrying these resistance genes.
None of the decreased taxa in this study (\textit{Arthrobacter globiformi}, \textit{Arthobacter} sp. Leaf69, \textit{Mycolicibacterium tusciae}, \textit{Microcoleus vaginatus}, \textit{Oscillatoria nigro-viridis}, \textit{Ramlibacter sp. Leaf400}) are known to carry $\beta$-lactam resistance genes, although one $\beta$-lactam resistance gene \textit{estA} has been identified in \textit{Arthrobacter nitroguajacolicus} Rü61a and several have been identified in the plasmidome of \textit{Mycolicibacterium} spp.

\todo[color=red!20]{Ed: Do you have any ideas for how I could expand on this section? Any other ties that you can see re: $\beta$-lactam resistance genes decreasing with macrolide exposure?}

\section{Policy implications}

The management of antibiotic concentrations in the environment is a shared responsibility between governments and antibiotic manufacturers:
Federal regulatory agencies must define acceptable levels for specific antibiotic residues in terrestrial and aquatic ecosystems, and manufacturers must develop and implement best practices to comply with these regulations.
Under the One Health framework, these acceptable levels should be decided based upon their potential effects on human, animal, and environmental health.

In Canada, the environmental risks of pharmaceuticals are jointly assessed by Health Canada and Environment and Climate Change Canada under the Canadian Environmental Protection Act of 1999 \parencite{Lee.2019}.
In 1994, Canada began to screen all new substances that were not in the Domestic Substances List (a list of substances that were already in use between 1984 to 1986) for their environmental toxicity, but existing substances, including many antibiotics, were not subject to the same testing \parencite{Lee.2019}.
In 2006, the Canadian government completed their safety evaluations of approximately 23,000 existing substances that were not previously analyzed:
Erythromycin, while determined to be persistent, was not determined to be inherently toxic to aquatic organisms or to meet the environmental criteria for categorization as a priority substance (\url{https://open.canada.ca/data/en/dataset/1d946396-cf9a-4fa1-8942-4541063bfba4}).

\todo[color=blue!20]{
  The only Canadian resource that I could access for evaluating the environmental impacts of pharmaceuticals was the Domestic Substances List, which doesn't include azithromycin or clarithromycin as these antibiotics came into use after 1986.
  Furthermore, the information on erythromycin is quite limited --- no info on PNECs or any quantitative metrics to manage concentrations in the environment.
  Are you aware of any Canadian lists of PNECs or equivalents?
  If not, I might have to leave the Canadian information as stated above and then move onto European efforts for which more information is available.
}

% In 2006, the Committee for Medicinal Products for Human Use (CHMP) within the European Medicines Agency published their guidelines for assessing the environmental health risks of pharmaceuticals in the environment \parencite{CommitteeforMedicinalProductsforHumanUse.2006}.
% The CHMP outlined a strategy for assessing the risks of pharmaceuticals to environmental health, which involves calculating PNEC values for...

% In this study, we explored the effects of long-term antibiotic pollution of soil on the bacterial community, resistome, and mobilome using two different doses of macrolide antibiotics:
% a low, environmentally realistic dose (0.1 mg kg-1) and a high, environmentally unrealistic dose (10 mg kg\textsuperscript{-1}).
% We noted that, while these doses were determined based upon surveys of macrolide antibiotics in municipal biosolids, there are some environments where macrolide antibiotic concentrations may reach or exceed the 10 mg kg\textsuperscript{-1} dose.
% At this high dose in soil, macrolide antibiotics promote antibiotic resistance by increasing the diversity of antibiotic resistance genes and mobile genetic elements and alter the composition of the soil bacterial resistome and mobilome.
% Given these insights, we must consider if existing recommendations for managing macrolide antibiotic concentrations in the environment are adequate to safeguard against the development of antibiotic resistance.

% Several metrics for calculating acceptable levels of antibiotic concentrations in the environment have been proposed \parencite{CommitteeforMedicinalProductsforHumanUse.2006, BengtssonPalme.2016, LePage.2017}, but these metrics are not always in-agreement with each other.
% In these cases, it has been proposed to consider the lowest no-effect concentration as the environmental threshold beyond which action should be taken to protect ecological health and mitigate antibiotic resistance \parencite{Tell.2019}.

\section{Strengths, limitations, and recommendations}

In this study, we investigated the effects of long-term, repeated exposure of macrolide antibiotics on the soil bacterial community, resistome, and mobilome using sequencing-based methods.
By using sequencing-based methods rather than culture-based methods, we were able to probe the uncultivated majority of bacteria residing in soil \parencite{Whitman.1998}.
Instead of using quantitative PCR to investigate the abundances of antibiotic resistance genes, mobile genetic elements, bacterial taxa, and integron gene cassettes, we deployed compositional data analysis techniques to obtain relative abundances of these features from our three sequencing datasets, and we analyzed the alpha and beta diversity of these features to compare richness and composition between our treatment groups.
By using sequencing-based methods over PCR for antibiotic resistance gene and mobile genetic element identification and quantification, we bypassed the need for PCR primers which would have restricted our ability to detect gene targets for which primers were not developed or were not available.
By performing two taxonomic analyses, one using a 16S rDNA amplicon dataset and another using a metagenomic DNA dataset, we identified a greater number of differentially abundant bacterial taxa for future investigation.

\subsection{Low read-merging for 16S rDNA paired-end sequences}

One potential concern with the results obtained from the 16S rDNA sequencing experiment was the low number of merged reads resulting from the DADA2 workflow (Supplementary Table \ref{supp-table:16S-sequencing-statistics}).
A low number of merged reads can result from poor sequence quality or from excessive trimming of the 3' ends of paired-end reads.
Our quality control analysis revealed overall good sequence quality for the 16S rDNA dataset, so it's likely that the Trimmomatic parameters that were used need to be re-adjusted to optimize the read-merging step while also discarding low-quality bases.
This loss of data could explain why Cyanobacteria were not identified as differentially abundant in the 16S rDNA dataset but were identified as differentially abundant in the metagenomic dataset.

\subsection{Environmental gene cassette sequencing}

The absence of a dose-dependent effect of macrolide antibiotic exposure on the integron gene cassette metagenome may be because macrolides do not induce the SOS response in bacteria, and therefore the transcription of integron integrases are not increased and recombination of gene cassettes does not occur at a greater frequency \parencite{Baharoglu.2010}.
Alternatively, because we sequenced environmental integron gene cassettes and didn't target class 1 integrons specifically, the environmental classes of integrons (of which there are hundreds) may have overwhelmed our gene cassette sequencing dataset, leaving few reads for class 1 integron gene cassettes, whose diversity and composition may have been affected by antibiotic exposure.
A future study investigating the response of the gene cassette metagenome of class 1 integrons to macrolide antibiotics could reveal trends that our compositional data analysis was not powered to detect.

\subsection{Identification of the resistome and mobilome hosts}

Next, the host bacteria responsible for the enrichment of antibiotic resistance genes and mobile genetic elements in this study were not determined.
We assumed that the bacteria that would be responsible for this increase would be revealed as differentially abundant in one of our taxonomic analyses, but it may be possible that the bacterial taxa that hosted these gene targets were not statistically differentially abundant yet were still sufficiently increased to enrich for antibiotic resistance.
These bacterial taxa could be revealed in the future by using a co-abundance network analysis to identify taxa with similar effect sizes to those of antibiotic resistance genes and mobile genetic elements, thereby allowing us to identify candidate taxa as hosts of these gene targets \parencite{Forsberg.2014}.

\subsection{Intermediate macrolide dose}

Finally, the absence of an intermediate concentration between our low dose (0.1 mg kg\textsuperscript{-1}) and our high dose (10 mg kg\textsuperscript{-1}) means that we were unable to precisely determine the 'threshold concentration' beyond which the soil resistome and mobilome were significantly affected by macrolide antibiotic exposure.
If this threshold concentration were to be within the range of 0.1 to 1 mg kg\textsuperscript{-1}, there could be cause-for-concern for some biosolids with a high macrolide antibiotic load to promote antibiotic resistance in soil.
Furthermore, an intermediate concentration of macrolides (1 mg L\textsuperscript{-1}) is more likely to be observed in an anthropogenically polluted environment than the high dose --- a similar investigation to this present study at this intermediate concentration could reveal similar effects on the bacterial resistomes and mobilomes in other environments.
