\chapter{Discussion}

The purpose of this project was to investigate the effect of long-term macrolide antibiotic exposure on the soil bacterial community, resistome, and mobilome --- and more specifically, to determine if an environmentally realistic dose of antibiotics for a biosolids exposure scenario could promote clinically relevant antibiotic resistance in soil bacteria.
Human and environmental health are interconnected as described by the One Health framework, and antibiotic resistance in soil bacteria may affect antibiotic resistance in the human microbiome or in pathogens.

\section{Realistic antibiotic exposure does not affect the diversity or composition of the soil bacterial community, resistome, or mobilome}

Overall, no effect of environmentally realistic antibiotic exposure (low dose, 0.1 mg kg\textsuperscript{-1}) on the diversity or composition of the soil bacterial community, resistome, or mobilome was detected.
This dose is similar to what would be expected in soil following the land-application (1--10\% dw dw\textsuperscript{-1}) of municipal biosolids containing 95\textsuperscript{th} percentile concentrations of erythromycin, clarithromycin, and azithromycin antibiotics.
The absence of a treatment effect for any of these endpoints indicates that repeated annual application of biosolids in agriculture is unlikely to promote antibiotic resistance in agricultural soil at levels that would be of concern to human health.

However, three antibiotic resistance genes had increased relative abundances in response to antibiotic exposure at the low dose, and two of them are known to be associated with resistance to macrolide antibiotics (Figure \ref{fig:effect-sizes-sig-args}).
\textit{mexL} and \textit{mexP} are members of the \textit{mex} gene family which are components of chromosomally encoded efflux pumps in \textit{Pseudomonas} spp. \parencite{Mima.2005, Chuanchuen.2005}.
The \textit{mexL} gene encodes a repressor for \textit{mexJK} transcription, which are members of the efflux pump-encoding \textit{mexJK-OpmH} (triclosan resistance) and \textit{mexOprM} (macrolide, tetracycline resistance) operons \parencite{Chuanchuen.2005}.
The \textit{mexP} gene encodes the membrane fusion protein for the multi-drug efflux pump that is encoded by the \textit{MexPQ-OpmE} operon, which is known to confer resistance to several antibiotic drug classes including macrolides \parencite{Mima.2005}.
The other antibiotic resistance gene that was increased in the low dosed-soil was \textit{aac(6')-IIa}, which encodes an aminoglycoside acetyltransferase that is distributed among a variety of gram-negative pathogens, including \textit{Pseudomonas} spp., and is carried by plasmids and integrons \parencite{Shaw.1989, Partridge.2009}.
Only one mobile genetic element variant, \textit{tnpA}, was increased in the low-dosed soil (Figure \ref{fig:effect-sizes-sig-mges}).
The \textit{tnpA} gene, of which there are many variants, encodes a transposase which is involved in the mobilization of several bacterial mobile genetic elements, such as transposons and insertion sequences across a broad host range \parencite{Partridge.2018}.
Three bacterial taxa were increased and are discussed in Chapter \ref{section:fastidious-taxa}.

Despite the increased relative abundances of three antibiotic resistance genes, one mobile genetic element variant, and three bacterial taxa in the low-dosed soil --- the overall diversity and composition of the soil bacterial community, resistome, and mobilome was unchanged by the low dose as indicated by richness (Figure \ref{fig:chao1-richness}, Supplementary Figure \ref{supp-fig:chao1-richness-taxa}) and PCA ordination plots (Figure \ref{fig:pca-ordplots-args-mges-integrons}, Supplementary Figure \ref{supp-fig:pca-ordplots-taxa}).
The increased abundances of a few antibiotic resistance genes in the low-dosed soil and that of \textit{tnpA} could be due to the increased abundance of a particular taxon that is intrinsically resistant to macrolide antibiotics, such as the human pathogen \textit{Pseudomonas aeruginosa}, though no taxa that were both significantly increased and known to carry this assortment of genes were detected.
These results are in clear contrast to the high dose of macrolide antibiotics.

\section{Unrealistically high antibiotic exposure alters the diversity and composition of the soil bacterial resistome and mobilome}

At a high dose of macrolide antibiotics (10 mg kg\textsuperscript{-1}) --- approximately 100-fold greater than the concentrations of macrolides that would be expected to result from land-application of municipal biosolids --- significant effects on the diversity and composition of the soil bacterial resistome and mobilome were detected.
While this dose is considered unrealistically high for soil receiving biosolids with macrolide concentrations within an upper-realistic range (upper 95\textsuperscript{th} percentile), the maximum azithromycin concentration detected in the \cite{U.S.EnvironmentalProtectionAgency.2009} biosolids survey (5 mg kg\textsuperscript{-1}) was within an order-of-magnitude of the high dose used in this present study.
This high dose of macrolide antibiotics has also been detected in sediments surrounding macrolide antibiotic manufacturing facilities \parencite{GonzalezPlaza.2019}.
Therefore, while this dose is described as unrealistically high for a biosolids land-application scenario, comparable doses are certainly seen under other environmental contexts, and the findings of this experiment may be useful for predicting antibiotic resistance in other macrolide-contaminated environments.

The high dose of macrolide antibiotics significantly increased the number of unique antibiotic resistance genes that were detected within the soil metagenome (Figure \ref{fig:chao1-richness}a).
The majority of antibiotic resistance genes that were detected in the metagenome were detected within all three groups (control, low, and high), but approximately twice as many unique antibiotic resistance genes were detected within the high treatment group alone than the control and low groups alone (Supplementary Figure \ref{supp-fig:args-mges-compartment-size}a).
The most likely explanation for the increased number of unique antibiotic resistance genes in the high-dosed soil is selection or co-selection for genes that were below the detection limit for the control and low groups, and were raised above the limit of detection by high antibiotic exposure (but not significantly increased based upon the differential abundance analysis).

In addition, a high dose of macrolide antibiotics changed the composition of antibiotic resistance genes within the soil metagenome (Figure \ref{fig:pca-ordplots-args-mges-integrons}a), and this effect extended to when antibiotic resistance genes were grouped by several drug classes (Figure \ref{fig:pca-ordplots-drug-classes}).
These differences in composition indicated that the relative abundances of several antibiotic resistance genes and target drug classes were more similar within the high-dosed soil than in the control and low-dosed soil.
The altered composition of antibiotic resistance genes in the high-dosed soil was driven by increased relative abundances of 21 antibiotic resistance genes in the high dose --- only two of which are known to confer resistance to macrolide antibiotics, while 11 are known to confer resistance to non-macrolide, ribosome-targeting drug classes (aminoglycoside, phenicol, tetracycline) (Figure \ref{fig:effect-sizes-sig-args}).

Like macrolides, the aminoglycoside, phenicol, and tetracycline drug classes of antibiotics also target the bacterial ribosome, but bind to different locations within the ribosome than do the macrolides \parencite{Pyorala.2014, Lohsen.2019}.
Because the precise ribosomal targets of these drug classes are different to that of macrolides, cross-resistance of antibiotic resistance genes to macrolides and drug classes other than lincosamide and streptogramin B antibiotics (whose targets overlap with that of macrolides) is uncommon, except for antibiotic resistance genes that encode multi-drug efflux pumps such as the \textit{mex} gene family in \textit{Pseudomonas} spp.
The increased relative abundances of non-macrolide antibiotic resistance genes in the high-dosed soil strongly suggests co-selection via co-resistance rather than cross-resistance
(except for \textit{mexQ}), which could be facilitated by mobile genetic elements.

Co-resistance to different drug classes of antibiotics can occur due to the genetic linkage of antibiotic resistance on mobile genetic elements such as class 1 integrons, or when antibiotic resistance genes are carried within the same host \parencite{Pal.2015}.
Of the 21 increased antibiotic resistance genes in the unrealistically high-dosed soil metagenome, eight are known to be associated with class 1 integrons (\textit{sul1}, \textit{aac(3)-Ib}, \textit{aadA}, \textit{aadA15}, \textit{aadA22}, \textit{aadA24}, \textit{dfrA17}, \textit{dfrA15}), which agrees with the increased relative abundance of \textit{intI1} and \textit{qacE$\Delta$1} (a component of the 3' conserved sequence) in the high dose \parencite{Partridge.2009, Yan.2006, Herrero.2008}, and with previously reported quantitative PCR observations \parencite{Lau.2020}.
All of these antibiotic resistance genes have been detected in gram-negative human pathogens \parencite{Alcock.2020}.

Despite the increased relative abundances of several metagenomic antibiotic resistance genes that are known to be associated with gene cassettes (except \textit{sul1}, which is a member of the 3' conserved sequence), integron gene cassette richness (Figure \ref{fig:chao1-richness}c) and composition (Figure \ref{fig:pca-ordplots-args-mges-integrons}c) were unaffected by antibiotic exposure, as determined by the integron gene cassette sequence analysis.
The absence of a treatment effect of macrolide antibiotic exposure on gene cassette richness and composition may be due to how \textit{intI1} gene expression --- and therefore, gene cassette recombination --- is regulated in bacteria.

The transcription of \textit{intI1} is regulated by the bacterial SOS system which is "a coordinated response to DNA damage" that is present in most bacteria \parencite{Maslowska.2019}.
Antibiotic drug classes that damage DNA (e.g. fluoroquinolones, nitrofurans) or affect DNA synthesis (e.g. sulfonamides, trimethoprim) induce the bacterial SOS response, which increases the expression of \textit{intI1} and thereby triggers integron gene cassette recombination \parencite{Guerin.2009, Baharoglu.2010}.
Antibiotic drug classes that do not damage DNA, such as macrolides, likely do not induce the bacterial SOS response and therefore do not increase the expression of \textit{intI1}, which would otherwise trigger integron gene cassette recombination \parencite{Hastings.2004}.
Therefore, macrolide antibiotic exposure of soil bacteria may not trigger gene cassette recombination, but antibiotics that induce the SOS response in soil bacteria may alter the richness or composition of the gene cassette metagenome and should be investigated for these effects.

Alternatively, because environmental integron gene cassettes were sequenced and class 1 integrons were not specifically targeted in the integron gene cassette targeted amplicon sequencing, the environmental classes of integrons (of which there are hundreds) may have overwhelmed our gene cassette sequencing dataset, leaving few reads for class 1 integron gene cassettes, whose diversity and composition may have been affected by antibiotic exposure.
A future study investigating the response of the gene cassette metagenome of class 1 integrons to macrolide antibiotics could reveal differences that our compositional data analysis was not powered to detect.

Of the remaining non-cassette-associated metagenomic antibiotic resistance genes that were increased in the high-dosed soil, five are known to be carried on plasmids in human pathogens (\textit{sul2}, \textit{aac(6')-Ib7}, \textit{pp-flo}, \textit{mphE}, \textit{ant(3'')-IIa}), \textit{tet(33)} is carried by the insertion sequence IS6100, and \textit{aph(3’’)-Ib} is carried on several mobile genetic elements including plasmids and transposons \parencite{Alcock.2020, Tauch.2002}.
The remaining genes with increased relative abundances are known to be chromosomally-encoded in \textit{Pseudomonas} spp. (\textit{aph(3’)-IIb}, \textit{oprN}, \textit{mexH}, \textit{triC}, \textit{mexQ}) or in \textit{Burkholderia} spp. (\textit{opcM}) \parencite{Hachler.1996, Mesaros.2007, Mima.2005, Mima.2007, Burns.1996}.
All of these remaining genes with the exception of \textit{aph(3')-IIb}, an aminoglycoside phosphotransferase, encode components of antibiotic efflux pumps.

Overall, of the 21 increased antibiotic resistance genes in the high-dosed soil, 15 are known to be carried by mobile genetic elements and all are known to be associated with human pathogens.
In considering the threat of these antibiotic resistance genes to human health, future research should be performed to determine if these genes reside within pathogenic bacteria in the soil or have the potential to be mobilized to human pathogens.

The high dose of antibiotics similarly increased the number of unique mobile genetic element variants (Figure \ref{fig:chao1-richness}b) and altered the composition of the mobilome (Figure \ref{fig:pca-ordplots-args-mges-integrons}).
More mobile genetic element variants were detected in the high-dosed soil group alone (\textit{n} = 119) than were shared between any combination of the other groups, suggesting that the high dose of macrolides raised many mobile genetic element variants over the limit of detection (Supplementary Figure \ref{supp-fig:args-mges-compartment-size}b).
Of the 23 mobile genetic element variants with increased relative abundances in the high-dosed soil (Figure \ref{fig:effect-sizes-sig-mges}), most were \textit{tnpA} variants (\textit{n} = 15), which suggests that some of the increased antibiotic resistance genes may have been mobilized by transposons or insertion sequences.

The exact mechanism of co-selection of macrolide and non-macrolide antibiotic resistance genes in the high-dosed soil could not be elucidated.
However, because antibiotic resistance genes that are known to be associated with several types of mobile genetic elements were increased, and several mobile genetic element variants were increased, it's plausible that multiple co-selection processes were active in the high-dosed soil simultaneously.

\section{Antibiotic exposure enriches for fastidious taxa} \label{section:fastidious-taxa}

In this study, increased relative abundances of three bacterial taxa in the low-dosed soil and two taxa in high-dosed soil were detected (Table \ref{table:effect-sizes-sig-taxa}).
For the low-dosed soil, the effect sizes for two of the three increased taxa were relatively low ($W$ $<$ 5), but an unknown \textit{Acidobacteria Subgroup 6} taxon was over 45-fold more abundant in the low-dosed soil than in the control.
This taxon was present in both antibiotic-treated groups but not in the control soil.

Acidobacteria are largely uncultivated, highly abundant bacteria in agricultural soil and play an important role in shaping the soil bacterial community through their decomposition of organic carbon \parencite{Solden.2016, Banerjee.2016, Banerjee.2016b}.
Furthermore, acidobacteria are a known reservoir of macrolide antibiotic resistance in urban surface waters through their expression of the \textit{erm} gene family, and have been reported to be increased in macrolide-polluted sediments, suggesting intrinsic macrolide resistance among some taxa \parencite{Yi.2019, Milakovic.2020}.
Conservatively, the unknown \textit{Acidobacteria Subgroup 6} taxon may represent a macrolide-resistant decomposer, but more speculatively, could represent an organism that is able to use macrolides as an alternative source of carbon.
Further studies would be required to investigate the macrolide biodegradation potential of this taxon.

For the high-dosed soil, the effect sizes for both increased taxa were high ($W$ $>$ 20) and both were identified as unknown \textit{Chloroflexi Gitt-GS-136} spp. (Table \ref{table:effect-sizes-sig-taxa}).
Chloroflexi are fastidious bacteria with diverse metabolisms and, like acidobacteria, are a known reservoir of macrolide resistance in the environment, though their overall role in environmental antibiotic resistance is still poorly understood \parencite{Gupta.2013, Islam.2019, Yi.2019}.
To our knowledge, this phylum has not been reported to carry any of the antibiotic resistance genes that were increased in this study, though the relationship between \textit{Chloroflexi} taxa and antibiotic resistance remains understudied \parencite{Razavi.2017}.

In the present study, it was assumed that the bacterial hosts of the antibiotic resistance genes that were increased at the high dose would be revealed as differentially abundant in at least one of the taxonomic analyses, but it's possible that the bacterial taxa that hosted these resistance genes were not significantly differentially abundant, yet were still sufficiently increased to enrich for antibiotic resistance.
These bacterial taxa could be revealed in a future co-abundance network analysis to identify taxa whose relative abundances are correlated with those of antibiotic resistance genes and mobile genetic elements, thereby allowing us to identify candidate taxa as hosts of these gene targets \parencite{Forsberg.2014}.

Although there was no significant effect of macrolide antibiotic exposure at either dose on the overall richness or composition of the soil bacterial community, some taxa are known to respond to exposure:
in a previous investigation of the persistence of macrolide antibiotics in soils that were annually exposed to a low or high dose of erythromycin, clarithromycin, and azithromycin for five years, or were left untreated, macrolide antibiotics were degraded more rapidly in the soils with an exposure history to macrolides than in the untreated control soil \parencite{Topp.2016}.
It is possible that \textit{Acidobacteria} or \textit{Chloroflexi} taxa may have played a role in the accelerated biodegradation of these macrolide antibiotics.

\section{Unrealistically high antibiotic exposure decreases relative abundances of cyanobacteria}

The only bacterial phylum that was differentially abundant in response to antibiotic exposure was \textit{Cyanobacteria} (Figure \ref{fig:effect-sizes-sig-phyla-metagenomic}).
The relative abundances of \textit{Cyanobacteria} sequence variants were decreased in the high-dosed soil but not in the low-dosed soil, and this effect was observed only in the metagenomic analysis and not in the 16S rDNA analysis (Supplementary Figure \ref{supp-fig:effect-sizes-sig-phyla-16S}).

Cyanobacteria have recently been considered as indicator species for antibiotic pollution of aquatic ecosystems due to their sensitivity to several drug classes of antibiotics \parencite{CommitteeforMedicinalProductsforHumanUse.2015, LePage.2017}, but this response is not uniform across all species and to all antibiotics \parencite{LePage.2017, Dias.2015}.
For example, the MIC of the cyanotoxin-producing cyanobacterium \textit{Microcystis aeruginosa} to $\beta$-lactam antibiotics can be as low as 0.1 mg L\textsuperscript{-1}, while the MICs of $\beta$-lactams for the tropical cyanobacteria \textit{Gloeocapsa} sp. and \textit{Chroococcidiopsis} sp. may be 100-fold greater \parencite{Dias.2015, Reynaud.1986}.

The decreased relative abundance of \textit{Cyanobacteria} sequence variants in the high-dosed soil of this present study suggests that the high dose, but not the low dose of macrolides is inhibitory to at least some cyanobacteria.
The minimum NOECs of azithromycin and erythromycin for growth inhibition of cyanobacteria were reported to be at-most 0.0015 and 0.0062 mg L\textsuperscript{-1}, which are approximately 20--70-fold lower than the concentration of macrolides in the low-dosed soil, and 1,600--6,700-fold lower than the concentration of macrolides in the high-dosed soil \parencite{LePage.2019}.
Therefore, the decreased abundance of \textit{Cyanobacteria} sequence variants in the high-dosed soil of this present study is in agreement with the known NOECs for erythromycin and azithromycin, but it should be noted that these values were determined for an aqueous environment and are likely different for soil.
Our inability to detect an effect of macrolide antibiotic exposure at the low dose may be due to a higher MIC for soil cyanobacteria or insufficient sensitivity to detect this effect using metagenomic sequencing.

The detection of this treatment effect in the high-dosed soil for only one of two taxonomic analyses may be due to differences in how abundances are calculated for each approach:
for the metagenomic analysis, metagenomic sequence reads were matched to a database of clade-specific marker genes and fold-coverages of these genes were obtained;
for the 16S rDNA analysis, amplicon sequence variants of 16S rDNA sequences were constructed and assigned taxonomy using a 16S rRNA gene database, and the number of times each variant was observed was counted.
Because 16S rRNA gene copy numbers are variable in bacteria, the relative abundances obtained from the 16S rDNA analysis are biased towards bacterial genomes with high copy numbers of the 16S rRNA gene \parencite{Kembel.2012}, whereas the metagenomic sequence analysis excluded the use of multicopy marker genes to assign taxonomy for this reason \parencite{Segata.2012}.

Another potential concern with the results obtained from the 16S rDNA sequencing experiment was the low number of merged reads resulting from the DADA2 workflow (Supplementary Table \ref{supp-table:sequencing-statistics-16S}).
A low number of merged reads can result from poor sequence quality or from excessive trimming of the 3’ ends of paired-end reads.
Our quality control analysis revealed overall good sequence quality for the 16S rDNA dataset, thus it is likely that the Trimmomatic parameters that were used need to be re-adjusted to optimize the read-merging step while also discarding low-quality bases.
This loss of data could explain why \textit{Cyanobacteria} sequence variants were not identified as differentially abundant in the 16S rDNA sequence analysis but were identified as differentially abundant in the metagenomic sequence analysis.

\section{High antibiotic exposure decreases relative abundances of $\beta$-lactam resistance genes}

Of the seven antibiotic resistance genes that were decreased in response to macrolide antibiotic exposure (five in the high dose, two in the low dose), all were predicted to encode resistance to $\beta$-lactam antibiotics (Figure \ref{fig:effect-sizes-sig-args}).
$\beta$-lactam antibiotics are bactericidal against both gram-negative and gram-positive bacteria by inhibiting synthesis of the cell wall, thereby leading to lysis and cell death \parencite{Balsalobre.2019}.
The $\beta$-lactam drug class of antibiotics was among the first to be brought to the drug market with the discovery of penicillin in 1928 by Alexander Fleming \parencite{Fleming.1929}.
The subsequent industrialized production and mass consumption of penicillins by the mid-1940's has resulted in increased acquired resistance to $\beta$-lactams, especially due to methicillin-resistant strains of \textit{Staphylococcus aureus} \parencite{PublicHealthAgencyofCanada.2020}.

$\beta$-lactam resistance genes are highly abundant in soil bacteria, even in the absence of anthropogenic antibiotic pollution, and over 90\% of these genes are encoded chromosomally \parencite{Dunivin.2019, vanGoethem.2018, Mindlin.2017}.
Of the $\beta$-lactam resistance genes that were decreased in relative abundance, two SHV-family $\beta$-lactamase encoding genes (\textit{bla}\textsubscript{SHV-71}, \textit{bla}\textsubscript{SHV-165}), one CTX-M $\beta$-lactamase (\textit{bla}\textsubscript{CTX-M-117}), one PEDO-family metallo-$\beta$-lactamase (\textit{bla}\textsubscript{PEDO-1}), and one \textit{ampC}-type $\beta$-lactamase (\textit{E. coli ampC}) were decreased in the high-dosed soil, while two TEM-family $\beta$-lactamase encoding genes (\textit{bla}\textsubscript{TEM-1}, \textit{bla}\textsubscript{TEM-22}) were decreased in the low-dosed soil.
\textit{bla}\textsubscript{TEM-1} was the first plasmid-associated $\beta$-lactam resistance gene to be identified and has since spread throughout gram-negative pathogens (e.g. \textit{Acinetobacter baumanii}, \textit{E. coli}, \textit{Klebsiella pneumoniae}).
Other members of the TEM-family of $\beta$-lactamase genes, including \textit{bla}\textsubscript{TEM-22}, have a more narrow host range but confer resistance to extended-spectrum $\beta$-lactams (able to hydrolyze oximino-cephalosporins) \parencite{Bradford.2001, Garlet.1993}.

The most likely explanation for the decreased abundances of $\beta$-lactam resistance genes in the macrolide antibiotic-exposed soil is the decreased abundance of macrolide-susceptible bacteria carrying these resistance genes.
While the co-selection of several non-macrolide antibiotic resistance genes in the high-dosed soil may have been due to genetic linkage between macrolide and non-macrolide antibiotic resistance genes, it is possible that macrolide and $\beta$-lactam resistance genes were infrequently genetically linked in the soil bacteria that were sampled, and that the $\beta$-lactam-resistant bacteria were outcompeted by macrolide-resistant bacteria in the presence of macrolides.
None of the decreased taxa in this study (\textit{Arthrobacter globiformi}, \textit{Arthobacter} sp. Leaf69, \textit{Mycolicibacterium tusciae}, \textit{M. vaginatus}, \textit{O. nigro-viridis}, \textit{Ramlibacter sp. Leaf400}) are known to carry $\beta$-lactam resistance genes, although one $\beta$-lactam resistance gene \textit{estA} has been identified in \textit{Arthrobacter nitroguajacolicus} Rü61a and several have been identified in the plasmidome of \textit{Mycolicibacterium} spp.

\section{Policy implications}

There are currently no globally accepted standards for setting limits for pollutant levels in biosolids.
In Europe, a “precautionary principle”-based approach for managing pollutant concentrations in biosolids and biosolids-applied soils has been used;
the absence of toxicity and fate data for these pollutants has led to huge variability in limits for pollutants in biosolids-applied soils between individual European nations --- sometimes by 2–3 orders of magnitude \parencite{McCarthy.2015}.
In the United States, the United Kingdom, and Canada, a risk assessment-based approach for managing pollutant concentrations in biosolids has been preferred \parencite{McCarthy.2015}.
In the United States, limits for pollutant levels have been established by considering different exposure pathways to humans (e.g. crop consumption, groundwater contamination), and by using the conservative approach of considering risk to the most highly-exposed individuals in society \parencite{McCarthy.2015}.

In Canada, most provinces and territories have set limits for pollutants (mostly inorganic) in biosolids, though this has not always been the case:
as more data of pollutant concentrations in biosolids has become available, more pollutants have been added to lists of agents requiring further research --- including antibiotics \parencite{Sabourin.2012, U.S.EnvironmentalProtectionAgency.2009, WaterEnvironmentAssociationofOntario.2010}.
Today, only two provinces in Canada (Québec and Nova Scotia) have set maximum limits for levels of organic compounds in biosolids, and no provinces or territories have set limits for antibiotics \parencite{McCarthy.2015}.

Due to the absence of experimental data on the effects of antibiotics in biosolids on soil bacteria, risk assessments based upon PNECs have been performed to guide policy decisions for managing antibiotic concentrations in biosolids \parencite{Jensen.2012, Eriksen.2009}.
Unfortunately, these risk assessments have faced many challenges, including the absence of existing toxicity and fate data for antibiotics (especially in biosolids-amended soil), and a failure to consider the simultaneous exposure to other antibiotics (and other pollutants) present in biosolids \parencite{McCarthy.2015}.
Generally, environmental risk assessments of antibiotics do not measure the potential for selection of antibiotic resistance \parencite{Lee.2019}.
However, a risk assessment by the Norwegian Scientific Committee for Food Safety compared the predicted environmental concentrations of 37 antibiotics in biosolids-applied soil to the MIC values of these antibiotics for \textit{Escherichia coli} and \textit{E. faecium};
they concluded that antibiotic resistance is unlikely to be promoted at an application rate of 60 tons biosolids ha \textsuperscript{-1} soil for all tested antibiotics except for the fluoroquinolone antibiotic, ciprofloxacin \parencite{Eriksen.2009}.
However, \textit{E. coli} and \textit{E. faecium} are intrinsically resistant to many of the antibiotics that were tested in this risk assessment, including erythromycin, clarithromycin, and azithromycin, and therefore cannot be used to conclude that antibiotic resistance will not be selected for in other biosolids-exposed soil bacteria.

In this thesis, I investigated the effects of long-term, repeated exposure of macrolide antibiotics on the soil bacterial community, resistome, and mobilome using sequencing-based methods, and I determined that an environmentally realistic dose of macrolides for a biosolids exposure scenario is unlikely to change soil bacterial diversity or promote clinically relevant antibiotic resistance.
These results suggest that typical concentrations of macrolide antibiotics within Canadian municipal biosolids are unlikely to harm environmental and human health in the context of antibiotic resistance.
However, the absence of an intermediate concentration between the low dose (0.1 mg kg\textsuperscript{-1}) and the high dose (10 mg kg\textsuperscript{-1}) means that I was unable to precisely determine the 'threshold concentration' beyond which the soil resistome and mobilome were significantly affected by macrolide antibiotic exposure for the soils used in this study.
If this threshold concentration were to be within the range of 0.1 to 1 mg kg\textsuperscript{-1}, there could be cause-for-concern for some biosolids with a high macrolide antibiotic load to promote antibiotic resistance in soil.
Furthermore, an intermediate concentration of macrolides (1 mg L\textsuperscript{-1}) is more likely to be observed in other anthropogenically polluted environments than the high dose \parencite{Bielen.2017}.

Overall, to protect human and environmental health, more data are required to establish acceptable limits for antibiotics in biosolids that are intended for agricultural use.
A similar investigation to this present study at intermediate concentrations of macrolides and in different contaminated environments may reveal similar treatment effects to those observed in the high-dosed bacteria of this study.
Future research is needed to elucidate the range of concentrations within which the soil resistome and mobilome are affected by macrolide antibiotic exposure.
