\chapter{Results}

DNA from untreated control soil and soil exposed to a low (0.1 mg kg$^{-1}$) or high (10 mg kg$^{-1}$) dose of macrolide antibiotics was used to generate three sequence datasets:
16S rDNA was sequenced to investigate the diversity and composition of the soil bacterial community,
total metagenomic DNA was sequenced to investigate the diversity and composition of the resistome and mobilome,
and integron gene cassettes were sequenced to investigate the diversity and composition of integron gene cassette open reading frames.

\section{Sequencing statistics} \label{section:sequencing-statistics}

Bacterial 16S rDNA amplicons were sequenced to investigate differences in soil bacterial community composition and diversity in response to antibiotic exposure.
16S rDNA MiSeq sequencing generated 6.21 M reads with an average of 50.7 $\pm$ 12.8 K unique reads per sample over 12 samples  (\textit{n} = 4 for each treatment group) (Supplementary Table \ref{supp-table:sequencing-statistics-16S}).
Following quality control, 6,837 to 72,287 merged reads were used for amplicon sequence variant assignment which resulted in 2,587 amplicon sequence variants.
One control sample was excluded from the 16S rDNA amplicon analysis due to a low number of merged reads resulting from the DADA2 workflow  (\textit{n} = 2,444).
The mean numbers of merged reads were not significantly different between treatment groups after removing this sample (one-way ANOVA, $F$ = 2.3, $p$ = 0.16).
The resulting 16S rDNA sequence reads were used to establish amplicon sequence variants which were taxonomically classified, and the number of reads for each bacterial taxon were obtained.

Next, metagenomic DNA was sequenced to investigate differences in the composition and diversity of the soil bacterial community, antibiotic resistance genes, and mobile genetic elements in response to antibiotic exposure.
Metagenomic NovaSeq sequencing generated 1.49 B reads with an average of 153 $\pm$ 18.7 M unique reads per sample over nine samples (\textit{n} = 3 for each treatment group) (Supplementary Table \ref{supp-table:sequencing-statistics-metagenomic}).
The resulting metagenomic sequence reads were i) taxonomically classified to obtain a second dataset of bacterial taxa abundances, and ii) mapped to antibiotic resistance gene and mobile genetic elements to obtain abundances for these antibiotic resistance determinants.

Finally, integron gene cassette amplicons were sequenced to investigate differences in the composition and diversity of gene cassette open reading frames in response to antibiotic exposure.
Integron gene cassette HiSeq sequencing generated 9.83 Gb of reads with an average of 0.94 $\pm$ 0.18 M unique reads per sample over 24 samples (\textit{n} = 4 biological replicates, \textit{n} = 2 technical replicates) (Supplementary Table \ref{supp-table:sequencing-statistics-integron}).
These reads were assembled into 270,368 contigs which were then filtered to retain only 75,850 high-confidence gene cassettes (28\%).
These gene cassettes were predicted to contain 72,628 open reading frames of which 36,050 (48\%) were considered unique.
Integron gene cassette open reading frames were analyzed for antibiotic resistance genes at three different levels of confidence (low, moderate, high), and to assign COG functional categories.
Integron gene amplicons were mapped onto the unique open reading frames to obtain abundances for these putative protein-coding genes.

The abundances that were obtained from each of these sequencing datasets allowed us to investigate differences in soil bacterial community, antibiotic resistance gene, mobile genetic element, and integron gene cassette open reading frame composition and diversity in response to macrolide antibiotic exposure.

\section{Bacterial community composition and diversity}

The richness (one-way ANOVA $F$ = 0.78, $p$ = 0.50) and composition (PERMANOVA pseudo-$F$ = 1.0, $p$ = 0.41) of soil bacterial taxa were not significantly affected by antibiotic exposure (Supplementary Figure \ref{supp-fig:chao1-richness-taxa}; Supplementary Figure \ref{supp-fig:pca-ordplots-taxa}).
However, at the phylum level, the relative abundances of \textit{Cyanobacteria} sequence variants were decreased in the high-dosed soil (Holm-Bonferroni-adjusted $p$-value $<$ 0.001, $W$ = -5.7 $\pm$ 0.7) but not in the low-dosed soil, relative to the control.
This result was observed only in the metagenomic analysis (Figure \ref{fig:effect-sizes-sig-phyla-metagenomic}) and not in the 16S rDNA analysis (Supplementary Figure \ref{supp-fig:effect-sizes-sig-phyla-16S}).
This difference was driven by the cyanobacteria \textit{Microcoleus vaginatus} ($p$ $<$ 0.001, $W$ = -5.2 $\pm$ 0.8) and \textit{Oscillatoria nigro-viridis} ($p$ $<$ 0.001, $W$ = -5.3 $\pm$ 0.3) in the high-dosed soil.

Only three taxa were differentially abundant by over 10-fold relative to the control:
an unknown \textit{Acidobacteria Subgroup 6} sp. was increased by approximately 46-fold in the low-dosed soil ($W$ = 45.8 $\pm$ 3.8), and two unknown \textit{Chloroflexi Gitt-GS-136} spp. were increased by approximately 23-fold ($W$ = 23.0 $\pm$ 0.8) and 59-fold ($W$ = 58.6 $\pm$ 0.4) in the high-dosed soil (Table \ref{table:effect-sizes-sig-taxa}).
These differences were observed only in the 16S rDNA analysis.

Overall, five bacterial species were increased in response to antibiotic exposure (three in the low dose, two in the high dose) and six species were decreased (one in the low dose, five in the high dose) (Table \ref{table:effect-sizes-sig-taxa}).
The taxa that were differentially abundant in the low- and high-dosed soil did not overlap (were not in-common) between the 16S rDNA amplicon and metagenomic taxonomic datasets.

\begin{figure}[htpb]
	\centering
		\includegraphics[width=\textwidth]{figures/metagenomic/effect-sizes-sig-phyla-metagenomic_v02.pdf}
	\caption[Effect sizes (fold-changes) of differences in the relative abundances of bacterial phyla classified in the metagenomic analysis relative to the untreated control soil.]{
		Effect sizes (fold-changes) of differences in the relative abundances of bacterial phyla classified in the metagenomic analysis relative to the untreated control soil.
		Horizontal lines intersecting with circles are error bars, indicating the extent of Bonferroni-adjusted 95\% confidence intervals of effect sizes (\textit{n} = 3).
	}
	\label{fig:effect-sizes-sig-phyla-metagenomic}
\end{figure}

\begin{table}[htpb]
	\centering
		\includegraphics[width=\textwidth]{tables/16S/effect-sizes-sig-taxa_v03.pdf}
	\caption[Effect sizes (fold-changes) of differentially abundant soil bacterial taxa in response to macrolide antibiotic exposure at low (0.1 mg kg\textsuperscript{-1}) and high (10 mg kg\textsuperscript{-1}) doses.]{
		Effect sizes (fold-changes) of differentially abundant soil bacterial taxa in response to macrolide antibiotic exposure at low (0.1 mg kg\textsuperscript{-1}) and high (10 mg kg\textsuperscript{-1}) doses.
		Effect sizes are stated with 95\% Bonferonni-adjusted confidence intervals (CIs).
		Differential abundance analysis was performed using ANCOM-BC for the 16S rDNA analysis (16S) or metagenomic analysis (M).
		All $p$-values are Holm-Bonferroni-adjusted.
		No taxa were identified as differentially abundant by both analyses.
	}
	\label{table:effect-sizes-sig-taxa}
\end{table}

\section{Resistome and mobilome composition and diversity}

\subsection{Resistome}

A total of 583 unique antibiotic resistance genes were detected across the soil metagenomes.
High macrolide antibiotic exposure significantly increased the richness of total antibiotic resistance genes in agricultural soil (Tukey’s all-pairs test, $p$ $<$ 0.05) but no effect was observed for the low dose (Figure \ref{fig:chao1-richness}a; Supplementary Figure \ref{supp-fig:args-mges-compartment-size}).
Similarly, high exposure but not low exposure changed the composition of antibiotic resistance genes (PERMANOVA pseudo-$F$ = 1.49, $p$ $<$ 0.05, 999 permutations) (Figure \ref{fig:pca-ordplots-args-mges-integrons}a).
These differences in composition were largely driven by 21 increased antibiotic resistance genes in the high dosed soil ($p$ $<$ 0.05) (Figure \ref{fig:effect-sizes-sig-args}).
Only five antibiotic resistance genes were differentially abundant (two decreased, three increased) in the low-dosed soil and no resistance gene was differentially abundant in both treatment groups.

\begin{figure}[htpb]
	\centering
		\includegraphics[width=\textwidth]{figures/mixed/chao1-richness_v03.pdf}
	\caption[Richness (Chao1) of antibiotic resistance genes, mobile genetic elements, and integron gene cassette open reading frames.]{
		Richness (Chao1) of antibiotic resistance genes, mobile genetic elements, and integron gene cassette open reading frames.
		Richness was determined for \textbf{a}) metagenomic antibiotic resistance genes, \textbf{b}) metagenomic mobile genetic elements, and \textbf{c}) integron gene cassette open reading frames of antibiotic-exposed and -unexposed soil bacteria.
		Horizontal lines connect samples within the same treatment group for visual clarity.
		Statistically significant comparisons between the antibiotic-exposed and untreated control soil are displayed (Kruskal-Wallis test, $p$ $<$ 0.05).
	}
	\label{fig:chao1-richness}
\end{figure}

\begin{figure}[htpb]
	\centering
		\includegraphics[width=0.9\textwidth]{figures/mixed/pca-ordplots-args-mges-integrons_v04.pdf}
	\caption[PCA ordination plots (PC1, PC2) of the CLR-transformed relative abundances of \textbf{a}) metagenomic antibiotic resistance genes, \textbf{b}) metagenomic mobile genetic elements, and \textbf{c}) integron gene cassette open reading frames in antibiotic-exposed and -unexposed soil bacteria.]{
		PCA ordination plots (PC1, PC2) of the CLR-transformed relative abundances of \textbf{a}) metagenomic antibiotic resistance genes, \textbf{b}) metagenomic mobile genetic elements, and \textbf{c}) integron gene cassette open reading frames in antibiotic-exposed and -unexposed soil bacteria.
		PERMANOVA pseudo-$F$ and $p$-values with 999 permutations are displayed.
		Shaded areas correspond to 95\% confidence ellipses of treatment groups.
		Percentages of variance explained by each axis are displayed in the axis titles.
	}
	\label{fig:pca-ordplots-args-mges-integrons}
\end{figure}

\begin{figure}[htpb]
	\centering
		\includegraphics[width=0.8\textwidth]{figures/metagenomic/effect-sizes-sig-args_v07.pdf}
	\caption[\textbf{a}) Effect sizes (fold-changes) of differences in the relative abundances of antibiotic resistance genes in antibiotic-exposed soil metagenomes and \textbf{b}) their target drug classes relative to the untreated control soil (\textit{n} = 3).]{
		\textbf{a}) Effect sizes (fold-changes) of differences in the relative abundances of antibiotic resistance genes in antibiotic-exposed soil metagenomes and \textbf{b}) their target drug classes relative to the untreated control soil (\textit{n} = 3).
		\textbf{a}. Only the genes that were differentially abundant ($p$ $<$ 0.05) with an absolute effect size of at least 5 (vertical dashed bars), for either treatment group, are shown.
		Shaded circles represent genes whose abundances were significantly different from the untreated control soil and open circles represent abundances that were not significantly different.
		Horizontal lines intersecting with circles are error bars, indicating the extent of Bonferroni-adjusted 95\% confidence intervals of effect sizes.
		\textbf{b}. Light grey squares indicate that resistance to a beta-lactam antibiotic is predicted; dark grey squares indicate that resistance to a ribosome-targeting drug class is predicted; black squares indicate that resistance to a non-beta-lactam and non-ribosome-targeting drug class is predicted (other).
	}
	\label{fig:effect-sizes-sig-args}
\end{figure}

The 21 antibiotic resistance genes that had increased relative abundances in the high dosed soil were predicted to confer resistance to 11 different drug classes of antibiotics and triclosan (a biocide), especially aminoglycosides (\textit{n} = 10) and diaminopyrimidines (\textit{n} = 4) (Figure \ref{fig:increased-drug-classes}).
Sixteen of these antibiotic resistance genes were predicted to confer resistance to classes of antibiotics which, like macrolides, target the ribosome.
Only two of these increased antibiotic resistance genes were predicted to encode resistance to macrolides (\textit{mphE}, \textit{mexQ}).
The gene that had the greatest increase in relative abundance in response to high antibiotic exposure was the aminoglycoside resistance gene \textit{aph(3’’)-Ib} ($W$ = 22.9 $\pm$ 0.5, $p$ $<$ 0.05).

\begin{figure}[htpb]
	\centering
		\includegraphics[width=\textwidth]{figures/metagenomic/increased-drug-classes_v04.pdf}
	\caption[Counts of target drug classes for which resistance was predicted to be encoded by antibiotic resistance genes within the soil metagenome.]{
		Counts of target drug classes for which resistance was predicted to be encoded by antibiotic resistance genes within the soil metagenome.
		Only counts for antibiotic resistance genes that were enriched in response to macrolide antibiotic exposure are displayed ($p$ $<$ 0.05, \textit{n} = 3).
	}
	\label{fig:increased-drug-classes}
\end{figure}

Analysis of antibiotic resistance genes grouped by their target drug class indicated that the compositions of aminoglycoside, diaminopyrimidine, phenicol, tetracycline, lincosamide, and streptogramin resistance genes, but not macrolide resistance genes, were significantly altered in the high-dosed soil ($p$ $<$ 0.05, 999 permutations) (Figure \ref{fig:pca-ordplots-drug-classes}).
Of the three antibiotic resistance genes that had increased relative abundances in the low-dosed soil ($p$ $<$ 0.05), two were predicted to encode resistance to macrolide antibiotics (\textit{mexL}, $W$ = 5.6 $\pm$ 0.2; \textit{mexP}, $W$ = 5.2 $\pm$ 0.2) and one was predicted to encode resistance to aminoglycosides (\textit{aac(6’)-IIa}, $W$ = 4.0 $\pm$ 0.4). No antibiotic resistance genes had increased relative abundances in both doses.

\begin{figure}[htpb]
	\centering
		\includegraphics[width=\textwidth]{figures/metagenomic/pca-ordplots-drug-classes_v03.pdf}
	\caption[PCA ordination plots (PC1, PC2) of CLR-transformed relative abundances of antibiotic resistance genes in the untreated control soil and in the low- and high-dosed soil, grouped by their target drug class.]{
		PCA ordination plots (PC1, PC2) of CLR-transformed relative abundances of antibiotic resistance genes in the untreated control soil and in the low- and high-dosed soil, grouped by their target drug class.
		PERMANOVA pseudo-$F$ and $p$-values with 999 permutations are displayed.
		Shaded areas correspond to 95\% confidence ellipses of treatment groups.
		Percentages of variance explained by each axis are displayed in the axis titles.
	}
	\label{fig:pca-ordplots-drug-classes}
\end{figure}

Seven antibiotic resistance genes had significantly decreased relative abundances relative to the control soil: five in the high dose and two in the low dose ($p$ $<$ 0.05).
Interestingly, all seven of these resistance genes were predicted to encode beta-lactamases (Figure \ref{fig:effect-sizes-sig-args}).
\textit{bla}\textsubscript{SHV-71} ($W$ = -24.1 $\pm$ 0.3), \textit{bla}\textsubscript{SHV-165} ($W$ = -11.0 $\pm$ 0.3), \textit{bla}\textsubscript{CTX-M-117} ($W$ = -7.6 $\pm$ 0.5), \textit{E. coli} \textit{ampC} ($W$ = -5.7 $\pm$ 0.4), and \textit{bla}\textsubscript{PEDO-1} ($W$ = -4.4 $\pm$ 0.2) were decreased in the high dose, and \textit{bla}\textsubscript{TEM-1} ($W$ = -4.9 $\pm$ 0.3) and \textit{bla}\textsubscript{TEM-22} ($W$ = -4.1 $\pm$ 0.8) were decreased in the low dose.

\subsection{Mobilome}

In addition to antibiotic resistance genes, the composition and diversity of mobile genetic elements within the soil metagenome was investigated.
Overall, 398 unique mobile genetic element variants were detected across the soil metagenomes, including several transposases and insertion sequence elements (e.g IS91, IS26).
As observed with antibiotic resistance genes, the richness of mobile genetic elements was significantly increased in the soil metagenome (Tukey’s all-pairs test, $p$ $<$ 0.05) (Figure \ref{fig:chao1-richness}b; Supplementary Figure \ref{supp-fig:args-mges-compartment-size}), and the composition of mobile genetic elements was significantly affected by the high dose of macrolides (PERMANOVA pseudo-$F$ = 2.01, $p$ $<$ 0.05, 999 permutations) (Figure \ref{fig:pca-ordplots-args-mges-integrons}b).

This altered composition of mobile genetic elements in the high-dosed soil was largely driven by 23 mobile genetic element variants with increased relative abundances ($p$ $<$ 0.05) (Figure \ref{fig:effect-sizes-sig-mges}).
Of these 23 increased mobile genetic elements, 15 were identified as \textit{tnpA}, three as \textit{intI1}, three as \textit{qacE$\Delta$1}, one as IS91, and one as \textit{tnpAN} variants.
The maximum effect size of the mobile genetic element variants that were increased in the high dose was $W$ = 23.8 $\pm$ 0.1 for \textit{intI1} ($p$ $<$ 0.05).

\begin{figure}[htpb]
	\centering
		\includegraphics[width=\textwidth]{figures/metagenomic/effect-sizes-sig-mges_v03.pdf}
	\caption[Effect sizes (fold-changes) of differences in the relative abundances of mobile genetic elements (MGEs) in antibiotic-exposed soil metagenomes relative to the untreated control soil (\textit{n} = 3).]{
		Effect sizes (fold-changes) of differences in the relative abundances of mobile genetic elements (MGEs) in antibiotic-exposed soil metagenomes relative to the untreated control soil (\textit{n} = 3).
		Only the mobile genetic elements that were differentially abundant ($p$ $<$ 0.05) with an absolute effect size of at least 5 (vertical dashed bars), for either treatment group, are shown.
		The name of the moobile genetic element is shown on the left, and the GenBank accession number of the reference sequence's genome is shown on the right.
		Shaded circles represent mobile genetic elements whose abundances were significantly different from the untreated control soil and open circles represent abundances that were not significantly different.
		Horizontal lines intersecting with circles are error bars, indicating the extent of Bonferroni-adjusted 95\% confidence intervals of effect sizes.
	}
	\label{fig:effect-sizes-sig-mges}
\end{figure}

The only mobile genetic element variant with an increased relative abundance in the low-dosed soil was identified as \textit{tnpA} ($W$ = 6.0 $\pm$ 0.3, $p$ $<$ 0.05) (Figure \ref{fig:effect-sizes-sig-mges}).
Of the three mobile genetic element variants that were decreased in the low-dosed soil (IS91, \textit{n} = 2; \textit{tnpA}, \textit{n} = 1), one IS91 variant was similarly decreased in the high-dosed soil (low dose, $W$ = -5.8 $\pm$ 0.4; high dose, $W$ = -5.7 $\pm$ 0.4).
No other mobile genetic element variants were differentially abundant in both doses.

\section{Composition and diversity of open reading frames from integron gene cassettes}

Both the richness (one-way ANOVA $F$ = 0.05, $p$ = 0.95) and composition (PERMANOVA pseudo-$F$ = 1.0, $p$ = 0.42) of integron gene cassette open reading frames were unaffected by antibiotic exposure (Figure \ref{fig:chao1-richness}c; Figure \ref{fig:pca-ordplots-args-mges-integrons}c).
Overall, 370 open reading frames (1\%) were identified as differentially abundant relative to the untreated control soil ($p$ $<$ 0.05), and of these 370 open reading frames, more were differentially abundant in the high-dosed soil (\textit{n} = 246, 67\%) than the low-dosed soil (\textit{n} = 144, 39\%).

\begin{figure}[htpb]
	\centering
		\includegraphics[width=0.85\textwidth]{figures/integron/effect-sizes-sig-integron-orfs_v04.pdf}
	\caption[Effect sizes (fold-changes) of differences in the relative abundances of integron gene cassette open reading frames in antibiotic-exposed soil bacteria relative to the untreated control soil.]{
		Effect sizes (fold-changes) of differences in the relative abundances of integron gene cassette open reading frames in antibiotic-exposed soil bacteria relative to the untreated control soil.
		Only the open reading frames that were differentially abundant ($p$ $<$ 0.05, \textit{n} = 3) with an absolute effect size $\ge$ 10 (vertical dashed bars), for either treatment group, are shown.
		The open reading frame (ORF) ID is shown on the left and the assigned COG functional category on the inner-right (if available).
		The name and highest confidence level (low, moderate, high) of predicted antibiotic resistance genes are shown on the outer-right.
		Shaded circles represent open reading frames whose abundances were significantly different from the untreated control soil and open circles represent abundances that were not significantly different.
		Horizontal lines intersecting with circles are error bars, indicating the extent of Bonferroni-adjusted 95\% confidence intervals of effect sizes.
		COG functional categories are as follows:
		K = Transcription,
		O = Post-translational modification, protein turnoover, chaperones;
		S = Function unknown.
	}
	\label{fig:effect-sizes-sig-integron-orfs}
\end{figure}

In total, 60 to 2,997 unique open reading frames (0.2 to 8.4\%) were predicted to encode antibiotic resistance depending on the confidence level used (see Chapter \ref{section:cassette-sequence-analysis}).
For the antibiotic resistance genes predicted at each confidence level, the most frequently detected target drug class of antibiotic resistance genes was aminoglycoside and the most frequently detected drug resistance mechanism was antibiotic inactivation.
Depending on the confidence level, 1 to 17 putative antibiotic resistance genes had increased relative abundances in response to antibiotic exposure and 1 to 13 putative antibiotic resistance genes decreased ($p$ $<$ 0.05) (Supplementary Table \ref{supp-table:integron-args-drug-classes}).
However, no putative antibiotic resistance genes were increased or decreased in both treatment groups at any confidence level.

Integron gene cassette open reading frames were also assigned COG functional categories to investigate if macrolide exposure changed the overall function of the cassette metagenome.
Only 5,206 (15\%) unique open reading frames could be assigned a functional category, and of those, 2,053 (39\%) were assigned a functional category other than 'function unknown' (S) (Supplementary Figure \ref{supp-fig:percentages-cog-categories}).
The open reading frames that were assigned to functional category EK (E: amino acid transport and metabolism; K: transcription) had slightly increased relative abundances ($W$ = 3.4 $\pm$ 1.1, $p$ $<$ 0.05), and those assigned to category DJ (D: cell cycle control, mitosis and meiosis; J: translation, ribosomal structure and biogenesis) had slightly decreased relative abundances ($W$ = -3.6 $\pm$ 0.8, $p$ $<$ 0.05) in the soil bacteria exposed to a high dose of macrolide antibiotics, but only a few open reading frames were assigned to each of these categories (EK, \textit{n} = 3; DJ, \textit{n} = 2).
