\chapter{Supplementary Information}
\myappendices{Supplementary Information}

%-------------------------------------------------------------------------------
% Supplementary Figures
%-------------------------------------------------------------------------------

\begin{suppfigure}[H]
	\centering
		\includegraphics[width=\textwidth]{supp-figures/16S/effect-sizes-sig-phyla-16S_v01.pdf}
	\caption[Effect sizes of bacterial phyla classified in the metagenomic analysis.]{
		Effect sizes (fold-changes) of differences in the relative abundances of bacterial phyla classified in the 16S rDNA analysis relative to the untreated control soil (\textit{n} = 4 for antibiotic-exposed groups, \textit{n} = 3 for untreated control group).
		Horizontal lines intersecting with circles are error bars, indicating the extent of Bonferroni-adjusted 95\% confidence intervals of effect sizes.
	}
	\label{supp-fig:effect-sizes-sig-phyla-16S}
\end{suppfigure}

\begin{suppfigure}[H]
	\centering
		\includegraphics[width=0.8\textwidth]{supp-figures/mixed/chao1-richness-taxa_v02.pdf}
	\caption[Richness of bacterial taxa.]{
		Richness (Chao1) of bacterial taxa as classified by \textbf{a}) the 16S rDNA analysis or \textbf{b}) the metagenomic analysis in macrolide antibiotic-exposed and untreated control soil.
		There were no statistically significant differences between the antibiotic-exposed and -unexposed groups.
		\textsuperscript{\Cross} \textit{n} = 4 for the antibiotic-exposed groups, \textit{n} = 3 for the untreated control group.
	}
	\label{fig:chao1-richness-taxa}
\end{suppfigure}

\begin{suppfigure}[htpb]
	\centering
		\includegraphics[width=0.75\textwidth]{supp-figures/mixed/pca-ordplots-taxa-metagenome-16S_v02.pdf}
	\caption[PCA ordination plots of bacterial taxa.]{
		PCA ordination plots (PC1, PC2) of the CLR-transformed relative abundances of bacterial taxa as classified by \textbf{a}) the 16S rDNA analysis or \textbf{b}) the metagenomic analysis in macrolide antibiotic-exposed and untreated control soil.
		PERMANOVA pseudo-$F$ and $p$-values with 999 permutations are displayed.
		Shaded areas correspond to 95\% confidence ellipses of treatment groups.
		Percentages of variance explained by each axis are displayed in the axis titles.
	}
	\label{supp-fig:pca-ordplots-taxa}
\end{suppfigure}

\begin{suppfigure}[H]
	\centering
		\includegraphics[width=\textwidth]{supp-figures/metagenomic/set-compartments-args-mges_v03.pdf}
	\caption[Number of antibiotic resistance genes and mobile genetic elements detected within each compartment formed between the untreated control, low-, and high-dosed soil groups.]{
		Number of antibiotic resistance genes and mobile genetic elements detected within each compartment formed between the untreated control, low-, and high-dosed soil groups, arranged by compartment size.
		Shaded dots below the bar plots correspond to the compartment.
	}
	\label{supp-fig:args-mges-compartment-size}
\end{suppfigure}

\begin{suppfigure}[H]
	\centering
		\includegraphics[width=0.75\textwidth]{supp-figures/integron/percentages-cog-categories_v02.pdf}
	\caption[Prevalence of COG functional categories assigned to integron gene cassette open reading frames.]{
		Prevalence (in percentages) of COG functional categories among integron gene cassette open reading frames that were assigned a COG function category (\textit{n} = 5,206).
		Only COG functional categories with a prevalence over 0.5\% are shown.
		Y-axis is logarithmically scaled and begins at 0.1\% for visual clarity.
		\todo[color=blue!20]{Is it okay for the y-axis to begin at 0.1\% instead of 0\%?}
	}
	\label{supp-fig:percentages-cog-categories}
\end{suppfigure}

%-------------------------------------------------------------------------------
% Supplementary Tables
%-------------------------------------------------------------------------------

\begin{supptable}[H]
	\centering
		\includegraphics[width=\textwidth]{supp-tables/primers_v01.pdf}
	\caption[PCR primer sequences, annealing temperatures, and expected amplicon sizes for integron gene cassette and 16S rDNA PCR amplification.]{
		PCR primer sequences, annealing temperatures, and expected amplicon sizes for integron gene cassette and 16S rDNA PCR amplification.
    Red text indicates location of adapter overhang sequences.
		\todo{Increase font size for this table.}
	}
	\label{supp-table:primers}
\end{supptable}

\begin{supptable}[H]
	\centering
		\includegraphics[width=\textwidth]{supp-tables/16S/16S-sequencing-statistics_v01.pdf}
	\caption[Summary of sequencing statistics for the 16S rDNA sequence dataset.]{
		Summary of sequencing statistics for the 16S rDNA sequence dataset.
		\todo{Increase font size for this table.}
	}
	\label{supp-table:16S-sequencing-statistics}
\end{supptable}
